


% == from paper ========================================================
% Abstract 1st paper (AACR):
% Multiple myeloma involves early dissemination of malignant plasma
% cells across the bone marrow; however, the initial steps of
% dissemination remain unclear. Human bone marrow-derived mesenchymal
% stromal cells (hMSCs) stimulate myeloma cell expansion (e.g., IL-6)
% and simultaneously retain myeloma cells via chemokines (e.g., CXCL12)
% and adhesion factors. Hence, we hypothesized that the imbalance
% between cell division and retention drives dissemination. 

% We present an in vitro model using primary hMSCs co-cultured with
% INA-6 myeloma cells. Time-lapse microscopy revealed proliferation and
% attachment/detachment dynamics. Separation techniques (V-well adhesion
% assay and well plate sandwich centrifugation) were established to
% isolate MSC-interacting myeloma subpopulations that were characterized
% by RNAseq, cell viability and apoptosis. Results were correlated with
% gene expression data (n=837) and survival of myeloma patients (n=536). 

% On dispersed hMSCs, INA-6 saturate hMSC-surface before proliferating
% into large homotypic aggregates, from which single cells detached
% completely. On confluent hMSCs, aggregates were replaced by strong
% heterotypic hMSC-INA-6 interactions, which modulated apoptosis
% time-dependently. Only INA-6 daughter cells (nMA-INA6) detached from
% hMSCs by cell division but sustained adherence to hMSC-adhering mother
% cells (MA-INA6).

% Isolated nMA-INA6 indicated hMSC-autonomy through superior viability
% after IL­6 withdrawal and upregulation of proliferation-related genes.
% MA-INA6 upregulated adhesion and retention factors (CXCL12), that,
% intriguingly, were highly expressed in myeloma samples from patients
% with longer overall and progression-free survival, but their
% expression decreased in relapsed myeloma samples. 

% Altogether, in vitro dissemination of INA-6 is driven by detaching
% daughter cells after a cycle of hMSC-(re)attachment and proliferation,
% involving adhesion factors that represent a bone marrow-retentive
% phenotype with potential clinical relevance.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Summary 2nd paper (JOSS):
% plotastic addresses the challenges of transitioning from exploratory
% data analysis to hypothesis testing in Python's data science ecosystem.
% Bridging the gap between seaborn and pingouin, this library offers a
% unified environment for plotting and statistical analysis. It simplifies
% the workflow with a user-friendly syntax and seamless integration with
% familiar seaborn parameters (y, x, hue, row, col). Inspired by seaborn's
% consistency, plotastic utilizes a DataAnalysis object to intelligently
% pass parameters to pingouin statistical functions. The library
% systematically groups the data according to the needs of statistical
% tests and plots, conducts visualisation, analyses and supports extensive
% customization options. In essence, plotastic establishes a protocol for
% configuring statical analyses through plotting parameters. This approach
% streamlines the process, translating seaborn parameters into statistical
% terms, allowing researchers to focus on correct statistical testing and
% less about specific syntax and implementations.

% Statement of need 2nd paper (JOSS):
% Python's data science ecosystem provides powerful tools for both
% visualization and statistical testing. However, the transition from
% exploratory data analysis to hypothesis testing can be cumbersome,
% requiring users to switch between libraries and adapt to different
% syntaxes.

% seaborn has become a popular choice for plotting in Python, offering an
% intuitive interface. Its statistical functionality focuses on
% descriptive plots and bootstrapped confidence intervals (Waskom, 2021).
% The library pingouin offers an extensive set of statistical tests, but
% it lacks integration with common plotting capabilities (Vallat, 2018).
% statannotations integrates statistical testing with plot annotations,
% but uses a complex interface and is limited to pairwise comparisons
% (Charlier et al., 2022).

% plotastic addresses this gap by offering a unified environment for
% plotting and statistical analysis. With an emphasis on user-friendly
% syntax and integration with familiar seaborn parameters, it simplifies
% the process for users already comfortable seaborn. The library ensures a
% smooth workflow, from data import to hypothesis testing and
% visualization

% ======================================================================
% == Abstract
% ======================================================================

% ## Treat this like a section (\section would add a header)
\addcontentsline{toc}{section}{Summary / Zusammenfassung} % > Add to Table of Contents
\markboth{Summary / Zusammenfassung}{} % > Mark this section for headers



% == English ===========================================================
\renewcommand{\abstractname}{Summary}
\begin{abstract}
    \label{Summary}%
    \vspace{-\vquarter}
    This thesis integrates biomedical research and data science, focusing on
    an \textit{in vitro} model for studying myeloma cell dissemination and a
    Python-based tool, \texttt{plotastic}, for semi-automated analysis of
    multidimensional datasets. Two major challenges are approached: (1)
    understanding the steps of myeloma dissemination and (2) improving
    data analysis efficiency to address the complexity- and reproducibility
    bottlenecks currently present in biomedical research.

    In the experimental component, primary human mesenchymal stromal cells
    (hMSCs) are co-cultured with INA-6 myeloma cells to study their cell
    proliferation, attachment, and detachment. Time-lapse microscopy reveal that
    predominantly myeloma daughter cells detach from hMSCs after cell division.
    Novel separation techniques were developed to isolate myeloma subpopulations
    for further characterization by RNAseq, cell viability, and apoptosis
    assays. Adhesion and retention genes are upregulated by MSC adhering INA-6
    cells, which correlates with patient survival. Overall, this work provides
    insights into myeloma dissemination mechanisms and identifies genes that
    potentially counteract dissemination through adhesion, which could be
    relevant for the design of new therapeutics.

    To manage the complex data resulting from the \textit{in vitro} model, a
    Python-based software named \texttt{plotastic} was developed that
    streamlines analysis and visualization of multidimensional datasets.
    \texttt{plotastic} is built on the idea that statistical analyses are
    performed based on how the data is visualized. This approach simplifies data
    analysis and semi-automates it in a standardized statistical protocol. The
    thesis becomes a case study as it reflects on the application of
    \texttt{plotastic} to the \textit{in vitro} model, demonstrating how the
    software facilitates rapid adjustments and refinements in data analysis and
    presentation. Such efficiency could be crucial for handling semi-big
    datasets transparently, which \dashed{despite being managable} are complex
    enough to complicate analysis and reproducibility.

    Together, this thesis illustrates the synergy between experimental
    methodologies and new data analysis tools. The \textit{in vitro} model
    provides a robust platform for studying myeloma dissemination, while
    \texttt{plotastic} addresses the need for efficient data analysis. Combined,
    they provide an approach for handling complex cell biological experiments
    and could advance both cancer biology and other research practices by
    supporting the exploratory investigation of challenging phenomena while
    communicating results transparently.
\end{abstract}

\newpage


%  ## Switch to German for correct hyphenation
\selectlanguage{ngerman}

% == German ============================================================
\renewcommand{\abstractname}{Zusammenfassung}
\begin{abstract}
    \label{Zusammenfassung}%
    \vspace{-\vquarter}

    Diese Doktorarbeit kombiniert biomedizinische Forschung und
    Datenwissenschaften und fokussiert sich auf ein \textit{in vitro}-Modell zur
    Untersuchung der Dissemination von Myelomzellen sowie auf die
    Python-Software \texttt{plotastic} zur semi-automatisierten Analyse
    multidimensionaler Datensätze. Zwei Hauptfragen werden behandelt: (1) das
    Verständnis der Schritte der Myelomdissemination und (2) die
    Steigerung der Effizienz der Datenanalyse, um die in der biomedizinischen
    Forschung bestehenden Herausforderungen hinsichtlich Komplexität und
    Reproduzierbarkeit zu bewältigen.

    Im experimentellen Teil werden primäre humane mesenchymale Stromazellen
    (hMSCs) mit INA-6-Myelomzellen kokultiviert, um ihre Proliferation,
    Anhaftung und Ablösung zu untersuchen. Zeitraffer-Mikroskopie zeigt, dass
    sich überwiegend Myelom-Tochterzellen nach Zellteilung von hMSCs ablösen.
    Neue Trennmethoden wurden entwickelt, um Myelom-Subpopulationen für weitere
    Charakterisierungen durch RNAseq, Zellviabilitäts- und Apoptose-Assays zu
    isolieren. Adhäsions- und Retentionsgene werden von MSC-adhärierenden INA-6
    hochreguliert, was mit Patientenüberleben korreliert. Insgesamt bietet
    diese Arbeit Einblicke in die Mechanismen der Myelomdissemination und
    identifiziert Gene, die potenziell die Dissemination durch Adhäsion
    behindern könnten, was für die Entwicklung neuer Therapien von Bedeutung
    sein könnte.
    
    Zur Handhabung der komplexen Daten, die bei dem \textit{in
    vitro}-Modell entstanden sind, wurde eine Python-basierte Software namens
    \texttt{plotastic} entwickelt, welche die Analyse und Visualisierung der
    multidimensionaler Datensätze optimiert. \texttt{plotastic} basiert auf dem Prinzip,
    dass statistische Analysen abhängig davon durchgeführt werden, wie die Daten
    visualisiert werden. Dieser Ansatz vereinfacht die Datenanalyse nicht nur,
    sondern semi-automatisiert sie auch als standardisiertes statistisches
    Protokoll. Diese Arbeit fungiert als Fallstudie, da sie die Anwendung von
    \texttt{plotastic} auf das \textit{in vitro}-Modell beleuchtet und zeigt,
    wie die Software schnelle Anpassungen und Verfeinerungen in der Datenanalyse
    und -präsentation ermöglicht. Eine solche Effizienz könnte entscheidend für
    den transparenten Umgang mit semi-großen Datenmengen sein, die trotz ihrer
    Handhabbarkeit komplex genug sind, um die Analyse und Reproduzierbarkeit zu
    erschweren.
    
    Zusammenfassend zeigt diese Dissertation die Synergie zwischen
    experimentellen Methoden und neuen Werkzeugen der Datenanalyse. Das
    \textit{in vitro}-Modell bietet eine robuste Plattform für die Untersuchung
    der Myelomdissemination, während \texttt{plotastic} den Bedarf an
    effizienter Datenanalyse unterstützt. Zusammen stellen sie einen Ansatz für
    die Bewältigung komplexer zellbiologischer Experimente dar, und könnten
    sowohl die Krebsbiologie als auch andere Forschungspraktiken förden, indem
    sie die explorative Untersuchung anspruchsvoller
    Phänomene unterstützen und dabei Ergebnisse transparent kommunizieren.


\end{abstract}

% ## Switch back to English
\selectlanguage{english}