



% ======================================================================
% == Chapter 1
% ======================================================================
\unnsection{Chapter 1: Modelling Myeloma Dissemination \textit{in vitro}}
\vspace{-\baselineskip} % > Remove space made by empty lines

% ## Reset reference counters of figs, tabs, so each chapter starts at 1
\setcounter{figure}{0}
\setcounter{table}{0}

% Title: Keep it Together: Modelling Myeloma Dissemination in vitro with
% hMSC- Interacting Subpopulations of INA-6 Cells and their
% Aggregation/Detachment Dynamics 

% ======================================================================
% == Abstract
% ======================================================================



% > First argument is the label
% > Second argument is the header name
% > Third argument is the abstract text
\customabstract{c1:abstract}{Abstract}{ %
    Multiple myeloma involves early dissemination of malignant plasma cells
    across the bone marrow; however, the initial steps of dissemination remain
    unclear. Human bone marrow- derived mesenchymal stromal cells (hMSCs)
    stimulate myeloma cell expansion (e.g., IL-6) and simultaneously retain
    myeloma cells via chemokines (e.g., CXCL12) and adhesion factors. Hence, we
    hypothesized that the imbalance between cell division and retention drives
    dissemination. We present an \textit{in vitro} model using primary hMSCs co-cultured
    with INA-6 myeloma cells. Time-lapse microscopy revealed proliferation and
    attachment/detachment dynamics. Separation techniques (V-well adhesion assay
    and well plate sandwich centrifugation) were established to isolate
    MSC-interacting myeloma subpopulations that were characterized by RNAseq,
    cell viability and apoptosis. Results were correlated with gene expression
    data ($n=837$) and survival of myeloma patients ($n=536$). On dispersed
    hMSCs, INA-6 saturate hMSC-surface before proliferating into large homotypic
    aggregates, from which single cells detached completely. On confluent hMSCs,
    aggregates were replaced by strong heterotypic hMSC-INA-6 interactions,
    which modulated apoptosis time-dependently. Only INA-6 daughter cells
    (\nMAina) detached from hMSCs by cell division but sustained adherence to
    hMSC-adhering mother cells (\MAina). Isolated \nMAina indicated
    hMSC-autonomy through superior viability after IL-6 withdrawal and
    upregulation of proliferation-related genes. \MAina upregulated adhesion and
    retention factors (CXCL12), that, intriguingly, were highly expressed in
    myeloma samples from patients with longer overall and progression-free
    survival, but their expression decreased in relapsed myeloma samples.
    Altogether, \textit{in vitro} dissemination of INA-6 is driven by detaching daughter
    cells after a cycle of hMSC-(re)attachment and proliferation, involving
    adhesion factors that represent a bone marrow-retentive phenotype with
    potential clinical relevance. %
}

% ## Statement of Significance
\subsection*{Statement of Significance}
Novel methods describe \textit{in vitro} dissemination of myeloma cells as detachment of
daughter cells after cell division. Myeloma adhesion genes were identified that
counteract \textit{in vitro} detachment with potential clinical relevance.
\newpage


% ======================================================================
% == Introduction
% ======================================================================
% \unnsubsection{C1:Introduction}{Introduction}
\unnsubsection{Introduction}\label{C1:introduction} % > Require unique label
\ %
Multiple myeloma arises from clonal expansion of malignant plasma cells in the
bone marrow (BM). At diagnosis, myeloma cells have disseminated to multiple
sites in the skeleton and, in some cases, to “virtually any tissue”
\cite{bladeExtramedullaryDiseaseMultiple2022,rajkumarInternationalMyelomaWorking2014}.
However, the mechanism through which myeloma cells initially disseminate remains
unclear. Dissemination is a multistep process involving invasion, intravasation,
intravascular arrest, extravasation, and colonization
\cite{zeissigTumourDisseminationMultiple2020}. To initiate dissemination,
myeloma cells overcome adhesion, retention, and dependency on the BM
microenvironment, which could involve the loss of adhesion factors such as CD138
\cite{akhmetzyanovaDynamicCD138Surface2020,garcia-ortizRoleTumorMicroenvironment2021}.
BM retention is mediated by multiple factors: First, chemokines (CXCL12 and
CXCL8) produced by mesenchymal stromal cells (MSCs), which attract plasma cells
and prime their cytoskeleton and integrins for adhesion
\cite{aggarwalChemokinesMultipleMyeloma2006,alsayedMechanismsRegulationCXCR42007}.
Second, myeloma cells must overcome the anchorage and physical boundaries of the
extracellular matrix (ECM), consisting of e.g. fibronectin, collagens, and
proteoglycans such as decorin
\cite{huDecorinmediatedSuppressionTumorigenesis2021,
    huangHigherDecorinLevels2015,katzAdhesionMoleculesLifelines2010,
    kiblerAdhesiveInteractionsHuman1998}. Simultaneously, ECM provides signals
inducing myeloma cell cycle arrest or progression the cell cycle
\cite{huDecorinmediatedSuppressionTumorigenesis2021,katzAdhesionMoleculesLifelines2010}.
\ %
ECM is also prone to degradation, which is common in several osteotropic
cancers, and is the cause of osteolytic bone disease. This is driven by a
‘vicious cycle' that maximizes bone destruction by extracting growth factors
(EGF and TGF-$\beta$) that are stored in calcified tissues
\cite{glaveyProteomicCharacterizationHuman2017}. Third, direct contact with MSCs
physically anchors myeloma cells to the BM
\cite{zeissigTumourDisseminationMultiple2020,sanz-rodriguezCharacterizationVLA4dependentMyeloma1999}.
Fourth, to disseminate to distant sites, myeloma cells require, at least
partially, independence from essential growth and survival signals provided by
MSCs in the form of soluble factors or cell adhesion signaling
\cite{garcia-ortizRoleTumorMicroenvironment2021,
    chatterjeePresenceBoneMarrow2002, hideshimaUnderstandingMultipleMyeloma2007}.
For example, the VLA4 (Myeloma)–VCAM1 (MSC)-interface activates NF-$\kappa$B in
both myeloma and MSCs, inducing IL-6 expression in MSCs. The independence from
MSCs is then acquired through autocrine survival signaling
\cite{frassanitoAutocrineInterleukin6Production2001,
    urashimaCD40LigandTriggered1995}. In short, anchorage of myeloma cells to MSCs
or ECM is a ‘double-edged sword': adhesion counteracts dissemination, but also
presents signaling cues for growth, survival, and drug resistance
\cite{solimandoDrugResistanceMultiple2022}.


To address this ambiguity, we developed an \textit{in vitro} co-culture system
modeling diverse adhesion modalities to study dissemination, growth, and
survival of myeloma cells and hMSCs. Co-cultures of hMSCs and the myeloma cell
line INA-6 replicated tight interactions and aggregate growth, akin to
``microtumors'' in Ghobrial's metastasis concept
\cite{ghobrialMyelomaModelProcess2012}. We characterized the growth
conformations of hMSCs and INA-6 as homotypic aggregation \textit{vs.}
heterotypic hMSC adherence and their effects on myeloma cell survival. We
tracked INA-6 detachments from aggregates and hMSCs, thereby identifying a
potential ``disseminated'' subpopulation lacking strong adhesion. We developed
innovative techniques (V-well adhesion assay and well plate sandwich
centrifugation) to separate weakly and strongly adherent subpopulations for the
subsequent analysis of differential gene expression and cell survival. Notably,
our strategy resolves the differences in gene expression and growth behavior
between cells of one cell population in ``direct'' contact with MSCs. In
contrast, previous methods differentiated between ``direct'' and ``indirect''
cell-cell contact using transwell inserts
\cite{dziadowiczBoneMarrowStromaInduced2022}. To evaluate whether genes
mediating adhesion and growth characteristics of INA-6 were associated with
patient survival, we analyzed publicly available datasets
\cite{seckingerTargetExpressionGeneration2017b,seckingerCD38ImmunotherapeuticTarget2018}.


\newpage


% ======================================================================
% ======================================================================
% ======================================================================
% == METHODS

\unnsubsection{Materials and Methods}%
\label{C1:methods}%
%%%%%%%%%
See \apdxref{subapdx:methods} for a complete method list and description.


\subsubsection*{Ethics Statement}%
\label{C1:methods_ethics}%
Primary human MSCs were collected with the written informed consent of all
patients. The procedure was conducted in accordance with recognized ethical
guidelines (Helsinki Declaration) and approved by the local Ethics Committee of
the University of Würzburg (186/18).



\subsubsection*{Cultivation and Co-Culturing of primary hMSCs and INA-6}%
\label{C1:methods_cultivation}%
Primary human MSCs were obtained from the femoral head of 34 non-myeloma
patients (\apdxref{apdx:supplemental}{tab:S1}: 21 male and 13 female, mean age
68.9 ± 10.6) undergoing elective hip arthroplasty. The INA-6 cell line
(\textit{DSMZ Cat\# ACC-862}, \textit{RRID:CVCL\_5209},
\href{https://www.cellosaurus.org/CVCL_5209}{link}) was initially isolated from
a pleural effusion sample obtained from an 80-year-old male with multiple
myeloma \cite{burgerGp130RasMediated2001c,gramatzkiTwoNewInterleukin61994}.
hMSCs were not tested for mycoplasma, whereas stocks of INA-6 were tested in
this study (\apdxref{apdx:supplemental}{tab:S1}) using the \textit{Venor GeM
    OneStep} kit (Minerva Biolabs, Berlin, Germany). For each co-culture, hMSCs were
seeded \SI{24}{\hour} before INA-6 addition to generate the \ac{CM}. INA-6 cells
were washed with PBS, resuspended in MSC medium, and added to hMSCs so that the
co-culture comprised \SI{33}{\percent} (v/v) of CM gathered directly from the
respective hMSC donor. The co-cultures were not substituted for IL-6
\cite{chatterjeePresenceBoneMarrow2002}.

% 132 Cell Viability and Apoptosis Assay
% 133 Cell viability and apoptosis rates were measured using CellTiter-Glo Luminescent Cell
% 134 Viability Assay and Caspase-Glo 3/7 Assay, respectively (Promega GmbH, Mannheim,
% 135 Germany).



\subsubsection*{Cell Viability and Apoptosis Assay}%
\label{C1:methods_cellviability}%
Cell viability and apoptosis rates were measured using \textit{CellTiter-Glo Luminescent
    Cell Viability Assay} and \textit{Caspase-Glo 3/7 Assay}, respectively (Promega GmbH,
Mannheim, Germany).


% 136 Automated Fluorescence Microscopy
% 137 Microscopic images were acquired using an Axio Observer 7 (Zeiss) with a COLIBRI LED
% 138 light source and motorized stage top using 5x and 10x magnification. The tiled images had
% 139 an automatic 8-10% overlap and were not stitched.

\subsubsection*{Automated Fluorescence Microscopy}%
\label{C1:methods_microscopy}%
Microscopic images were acquired using an Axio Observer 7 (Zeiss) with a COLIBRI LED
light source and motorized stage top using 5x and 10x magnification. The tiled images had
an automatic \SIrange{8}{10}{\percent} overlap and were not stitched.


% 140 Live Cell Imaging
% 141 hMSCs (stained with PKH26) were placed into an ibidi Stage Top Incubation System and
% 142 equilibrated to 80% humidity and 5% CO2. INA-6 (2 × 103 cells/cm2) were added directly
% 143 before the start of acquisition. Brightfield and fluorescence images of up to 13 mm2 of the co-
% 144 culture area were acquired every 15 min for 63 h. Each event of interest was manually
% 145 analyzed and categorized into defined event parameters.

\subsubsection*{Live Cell Imaging}%
\label{C1:methods_livecell}%
hMSCs (stained with PKH26) were placed into an ibidi Stage Top Incubation System
and equilibrated to \SI{80}{\percent} humidity and \SI{5}{\percent}
CO\textsubscript{2}. INA-6 (2 $\times$ 10\textsuperscript{3}
cells/cm\textsuperscript{2}) were added directly before the start of
acquisition. Brightfield and fluorescence images of up to \SI{13}{\mm\squared}
of the co-culture area were acquired every \SI{15}{\minute} for \SI{63}{\hour}.
Each event of interest was manually analyzed and categorized into defined event
parameters.


% 146 V-well Adhesion Assay
% 147 INA-6 cells were arrested during mitosis by two treatments with thymidine, followed by
% 148 nocodazole. Arrested INA-6 were released and added to 96 V-well plates (104 cells/cm2) on
% 149 top of confluent hMSCs and adhered for 1-3 h. The co-culture was stained with calcein-AM
% 150 (Thermo Fisher Scientific, Darmstadt, Germany) before non-adherent INA-6 were pelleted
% 151 into the tip of the V-well (2000 rpm, 5-10 min). MSC-adhering INA-6 cells were manually
% 152 detached by rapid pipetting. The pellet brightness was measured microscopically and the
% 153 pellet was isolated by pipetting.


\subsubsection*{V-well Adhesion Assay}%
\label{C1:methods_vwell}%
INA-6 cells were arrested during mitosis by two treatments with thymidine,
followed by nocodazole. Arrested INA-6 were released and added to 96 V-well
plates (10\textsuperscript{4} cells/cm\textsuperscript{2}) on top of confluent
hMSCs and adhered for \SIrange{1}{3}{\hour}. The co-culture was stained with
calcein-AM (Thermo Fisher Scientific, Darmstadt, Germany) before non-adherent
INA-6 were pelleted into the tip of the V-well (\SI{555}{$g$},
\SIrange{5}{10}{\minute}). MSC-adhering INA-6 cells were manually detached by
rapid pipetting. The pellet brightness was measured microscopically and the
pellet was isolated by pipetting.

% 154 Cell Cycle Profiling by Image Cytometry
% 155 Isolated INA-6 cells were fixed in 70% ice-cold ethanol, washed, resuspended in PBS,
% 156 distributed in 96-well plates, and stained with Hoechst 33342. The plates were scanned at 5x
% 157 magnification. A pre-trained convolutional neural network (Intellesis, Zeiss) was fine-tuned to
% 158 segment the scans into single nuclei and exclude fragmented nuclei. Nuclei were filtered to
% 159 exclude extremes of size roundness. The G0/G1 frequency was determined by Gaussian
% 160 curve fitting.

\subsubsection*{Cell Cycle Profiling by Image Cytometry}%
\label{C1:methods_cellcycle}%
Isolated INA-6 cells were fixed in \SI{70}{\percent} ice-cold ethanol, washed, resuspended in PBS,
distributed in 96-well plates, and stained with Hoechst 33342. The plates were scanned at 5x
magnification. A pre-trained convolutional neural network (Intellesis, Zeiss) was fine-tuned to
segment the scans into single nuclei and exclude fragmented nuclei. Nuclei were filtered to
exclude extremes of size and roundness. The G0/G1 frequency was determined by Gaussian
curve fitting.


\subsubsection*{\ac{WPSC}}%
\label{C1:methods_wpsc}%
hMSCs were grown to confluence in 96-well plates coated with collagen I (rat
tail; Corning, NY, USA). INA-6 cells were added and the cells were allowed to
adhere for \SI{24}{\hour}. A second plate (``catching plate'') was attached
upside down to the top of the co-culture plate. That ``well plate sandwich'' was
turned around and the content of the co-culture plate was centrifuged into the
catching plate three times (\SI{40}{\second} at \SI{110}{g}) while gently adding
\SI{30}{\micro\liter} of medium in between centrifugation steps.
Non-MSC-adhering INA-6 cells were collected from the catching plate, whereas
MSC-adhering INA-6 cells were isolated by digesting the co- culture with
accutase. For RNA sequencing (RNAseq), all samples were purified using anti-CD45
\ac{MACS} (Miltenyi Biotec B.V. \& Co. KG, Bergisch
Gladbach).

\subsubsection*{RNA Isolation}%
\label{C1:methods_rna}%
RNA was isolated using the \textit{NucleoSpin RNA II Purification Kit}
(Macherey-Nagel) according to the manufacturer's instructions. RNA was isolated
from INA-6 cells co-cultured with a unique hMSC donor ($n=5$ for RNA sequencing, $n=11$
for qPCR).

\subsubsection*{RNA sequencing, Differential Expression, and Functional Enrichment Analysis}%
\label{C1:methods_rnaseq}%
RNA sequencing (RNAseq) was performed at the Core Unit Systems Medicine,
University of Würzburg. mRNA was enriched with polyA beads. Fastq files were
aligned to the GRCh38 reference genome using \texttt{STAR}
(\textit{RRID:SCR\_004463},
\href{https://scicrunch.org/resolver/RRID:SCR_004463}{link}) and raw read
counts were generated using \texttt{HTseq} (\textit{RRID:SCR\_005514},
\href{https://scicrunch.org/resolver/SCR_005514}{link})
\cite{andersHTSeqPythonFramework2015,dobinSTARUltrafastUniversal2013,zerbinoEnsembl20182018}.
Differential gene expression was analyzed using \texttt{edgeR} in R (version
3.6.3) (\textit{RRID:SCR\_012802},
\href{https://scicrunch.org/resolver/SCR_012802}{link}). Functional
enrichment analysis was performed using \texttt{Metascape}
(\textit{RRID:SCR\_016620},
\href{https://scicrunch.org/resolver/SCR_016620}{link})
\cite{zhouMetascapeProvidesBiologistoriented2019}.


% 182 RT-qPCR
% 183 RNA (1 μg) was reverse transcribed using SuperScript IV reverse transcriptase (Thermo
% 184 Fisher Scientific). qPCR was performed using 10 μL GoTaq qPCR Master Mix (Promega),
% 185 1:10 diluted cDNA, and 5 pmol of primers obtained from Biomers.net or Qiagen
% 186 (Supplementary Table 3).
\subsubsection*{RT-qPCR}%
\label{C1:methods_rtqpcr}%
RNA (\SI{1}{\micro\gram}) was reverse transcribed using \textit{SuperScript IV
    reverse transcriptase} (Thermo Fisher Scientific). qPCR was performed using
\SI{10}{\micro\liter} \textit{GoTaq qPCR Master Mix} (Promega), \SI{1}{:10}
diluted cDNA, and \SI{5}{pmol} of primers obtained from Biomers.net or Qiagen
(\apdxref{apdx:supplemental}{tab:S3}).


% 187 Statistics
% 188 Inferential statistics were performed using Python (IPython, RRID:SCR_001658, link) (3.10)
% 189 packages pingouin (0.5.1) and statsmodels (0.14.0) (29, 30). The figures were plotted using
% 190 plotastic (0.0.1) (31). Normality (for n ≥ 4) and sphericity were ensured using Mauchly's and
% 191 Shapiro-Wilk tests, respectively. Data points were log10 transformed to convert the scale
% 192 from multiplicative to additive or to fulfill sphericity requirements. p-value = 0.05 > * > 0.01 >
% 193 ** > 10-3 > *** 10-4 > ****. P-values were either adjusted (p-adj) or not adjusted (p-unc) for
% 194 family wise error rate. Power calculations were not performed to determine the sample size.
\subsubsection*{Statistical Analysis}%
\label{C1:methods_statistics}%
Inferential statistics were performed using Python (IPython,
\textit{RRID:SCR\_001658},
\href{https://www.ncbi.nlm.nih.gov/sra?term=SCR_001658}{link}) (3.10) packages
\textit{pingouin} (0.5.1) and \texttt{statsmodels} (0.14.0)
\cite{vallatPingouinStatisticsPython2018,seaboldStatsmodelsEconometricStatistical2010}.
The figures were plotted using \texttt{plotastic} (0.0.1)
\cite{kuricPlotasticBridgingPlotting2024}. Normality (for \( n \geq 4 \)) and
sphericity were ensured using Mauchly's and Shapiro-Wilk tests, respectively.
Data points were Log\(_{10}\) transformed to convert the scale from
multiplicative to additive or to fulfill sphericity requirements. \( p = 0.05 >
\text{*} > 0.01 > \text{**} > 10^{-3} > \text{***}> 10^{-4} > \text{****} \).
$p$-values were either adjusted (\( p{\text{-adj}} \)) or not adjusted (\(
p{\text{-unc}} \)) for family wise error rate. Power calculations were not
performed to determine the sample size.


% 195 Patient Cohort, Analysis of Survival and Expression
% 196 Survival and gene expression data were obtained as previously described (21, 22) and are
% 197 available at the European Nucleotide Archive (ENA) under accession numbers PRJEB36223
% 198 and PRJEB37100. The expression level was categorized into “high” and “low” using maxstat
% 199 (Maximally selected Rank Statistics) thresholds (32).
% 200 Data Availability Statement
% 201 A detailed description of the methods is provided in the Supplementary Material section. Raw
% 202 tabular data and examples of analyses and videos are available in the github repository, link.
% 203 Raw RNAseq data are available from the NCBI Gene Expression Omnibus (GEO)
% 204 (RRID:SCR_005012, link) (GSE261423). Microscopy data are available at BioStudies
% 205 (EMBL-EBI) (RRID:SCR_004727, link) (S-BIAD1092).

\subsubsection*{Patient Cohort, Analysis of Survival and Expression}%
\label{C1:methods_patientcohort}%
Survival and gene expression data were obtained as previously described
\cite{seckingerTargetExpressionGeneration2017b,seckingerCD38ImmunotherapeuticTarget2018}
and are available at the European Nucleotide Archive (ENA) under accession
numbers PRJEB36223 and PRJEB37100. The expression level was categorized into
``high'' and ``low'' using \texttt{maxstat} (Maximally selected Rank Statistics)
thresholds \cite{hothornMaximallySelectedRank}.

% 200 Data Availability Statement
% 201 A detailed description of the methods is provided in the Supplementary Material section. Raw
% 202 tabular data and examples of analyses and videos are available in the github repository, link.
% 203 Raw RNAseq data are available from the NCBI Gene Expression Omnibus (GEO)
% 204 (RRID:SCR_005012, link) (GSE261423). Microscopy data are available at BioStudies
% 205 (EMBL-EBI) (RRID:SCR_004727, link) (S-BIAD1092).

\subsubsection*{Data Availability Statement}%
\label{C1:methods_dataavailability}%
A detailed description of the methods is provided in the Supplementary Material section. Raw
tabular data and examples of analyses and videos are available in the github repository, \href{https://github.com/markur4/Supplemental-INA-6-Subpopulations-and-Aggregation-Detachment-Dynamics}{link}.
Raw RNAseq data are available from the NCBI Gene Expression Omnibus (GEO)
(\textit{RRID:SCR\_005012}, \href{https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261423}{link}) (GSE261423). Microscopy data are available at BioStudies
(EMBL-EBI) (\textit{RRID:SCR\_004727}, \href{https://www.ebi.ac.uk/biostudies/bioimages/studies/S-BIAD1092?key=69bafe9c-74ff-492b-9e68-bd42655c4d1b}{link}) (S-BIAD1092).

\newpage





% ======================================================================
% ======================================================================
% ======================================================================
% == RESULTS


\unnsubsection{Results}\label{C1:results}%
%%%%%%
% 
% 
% ==  ==================================================================
% == MSC Saturation
% == ===================================================================
% 
\subsubsection*{\INA Cells Saturate hMSC-Interaction to Proliferate into Aggregates}%
\label{C1:results_saturation}%
hMSCs are isolated as a heterogeneous cell population. To analyze whether \INA
cells could adhere to every hMSC, we saturated hMSCs with \INA. A seeding ratio
of 1:4 (hMSC:\INA) resulted in the occupation of \SIplusminus{93}{6}{\percent}
of single hMSCs by one or more \INA cells within 24 hours after \INA addition,
escalating to \SI{6}{\percent} after \SI{48}{hours} (\autoref{fig:1}A, B).
Therefore, most hMSCs provide an interaction surface for \INA cells.

INA-6 exhibits homotypic aggregation when cultured alone, a phenomenon observed
in some freshly isolated myeloma samples (up to \SI{100}{cells} after
\SI{6}{hours}) \cite{kawanoHomotypicCellAggregations1991a,
    okunoVitroGrowthPattern1991}. Adding hMSCs at a 1:1 ratio led to smaller
aggregates after \SI{24}{hours} (size \SIrange{1}{5}{cells}), all of which were
distributed over \SIplusminus{52}{2}{\percent} of all hMSCs (\autoref{fig:1}A,
B). Intriguingly, \INA aggregation was notably absent when grown on confluent
hMSCs, and occurred only when heterotypic interactions were limited to 0.2 hMSCs
per \INA cell (\autoref{fig:1}C). We concluded that \INA cells prioritize
heterotypic over homotypic interactions.

To monitor the formation of such aggregates, we conducted live-cell imaging of
hMSC/\INA co-cultures for \SI{63}{hours}. We observed that \INA cells adhered
long after cytokinesis, constituting \SIplusminus{55}{12}{\percent} of all
homotypic interactions between \SI{13}{hours} and \SI{26}{hours}, increasing to
more than \SI{75}{\percent} for the remainder of the co-culture
(\autoref{fig:1}D). Therefore, homotypic \INA aggregates were mostly formed by
cell division.

% ======================================================================
\subsubsection*{Apoptosis of \INA Depends on Ratio Between Heterotypic and Homotypic Interaction}%
\label{C1:results_apoptosis}%
Although direct interaction with hMSCs has been shown to enhance myeloma cell
survival through NF-$\kappa$B signaling
\cite{hideshimaUnderstandingMultipleMyeloma2007}, the impact of aggregation on
myeloma cell viability during hMSC interaction remains unclear. To address this,
we measured the cell viability (ATP) and apoptosis rates of \INA cells growing
as homotypic aggregates compared to those in heterotypic interactions with hMSCs
by modulating hMSC density (\autoref{fig:1}E). To equalize the background
signaling caused by soluble MSC-derived factors, all cultures were incubated in
hMSC-conditioned medium and the results were normalized to \INA cells cultured
without direct hMSC contact (\autoref{fig:1}E,\,left).


\newpage

% ## Example:
% > \includeimage[scale][left bottom right top]{
% >     pathtopicture
% > }{caption}

% == Fig. 1 ============================================================


\includeimage[1][{\fmleft} 2.8in {\fmright} {\fmtop}]{
    FIGS/figures9_1.pdf
}\splitcaption[fig:1]{ %
    INA-6 growth conformations and survival on hMSCs. \tile{A} Interaction of
    INA-6 (green) with hMSCs (black, negative staining) at different INA-6
    densities (constant hMSC densities). \mbox{Scale\,bar\,=\,\SI{200}{\um}}.
    \tile{B} Frequency of single hMSCs (same as A) that are covered by INA-6 of
    varying group sizes. Technical replicates = three per datapoint; 100 single
    hMSCs were evaluated per technical replicate.
    \tile{C} Interaction of INA-6 with hMSCs at different hMSC densities (constant
    INA-6 densities). Scale bar = \SI{300}{\um}.
}{%
    \tile{D} Two types of homotypic
    interaction: Attachment after cell contact and sustained attachment of
    daughter cells after cell division. Datapoints represent one of four
    independent time-lapse recordings, each evaluating 116 interaction events.
    \tile{E} Effects of hMSC-density on the viability (ATP, top) and apoptosis
    (Caspase3/7 activity, bottom). INA-6:MSC ratio = 4:1; \mbox{Technical
        replicates = four per datapoint}; \tile{E\,left} Signals were measured in
    INA-6 washed off from hMSCs and normalized by INA-6 cultured in
    MSC-conditioned medium ($= \text{red line}$) ($n=4$). \tile{E\,right}
    Signals were measured in co-cultures and normalized by the sum of the
    signals measured in hMSC and INA-6 cultured separately ($= \text{red line}$)
    ($n=3$). \tile{Statistics} Paired t-test, two-factor RM- ANOVA. Datapoints
    represent independent co-cultures with hMSCs from three (A, B, D, E\,right),
    four (E\,left) unique donors. Confl.\,=\,Confluent. }


\INA viability (ATP) was not affected by the direct adhesion of hMSCs at any
density. However, apoptosis rates decreased over time
\omnibus{F}{2,6}{23.29}{-unc}{\scinot{1.49}{-3}} (Two-factor RM-ANOVA),
interacting significantly with MSC density
\omnibus{F}{4,12}{6.98}{-unc}{\scinot{3.83}{-3}} For example, 24 hours of
adhesion to confluent MSCs increased apoptosis rates by
\SIplusminus{1.46}{0.37}{fold}, while culturing \INA cells on dispersed hMSCs
(ratio 1:1) did not change the apoptosis rate (\SIplusminus{1.01}{0.26}{fold}).


We presumed that sensitive apoptotic cells might have been lost when harvesting
\INA cells from hMSCs. Hence, we measured survival parameters in the co-culture
and in hMSC and \INA cells cultured separately (\autoref{fig:1}E,\,right). We
defined MSC interaction effects when the survival measured in the co-culture
differed from the sum of the signals measured from \INA and hMSCs alone.
RM-ANOVA confirmed that adherence to confluent MSCs increased apoptosis rates of
\INA cells \SI{24}{hours} after adhesion and decreased after \SI{72}{hours}
\omnibus{F}{2,4}{26.86}{-unc}{\scinot{4.80}{-3}} (interaction between MSC
density and time, Two-factor RM-ANOVA), whereas \INA cells were unaffected when
grown on dispersed hMSCs.
In summary, the growth conformation of \INA cells, measured as the ratio between
homotypic aggregation and heterotypic MSC interactions, affected apoptosis rates
of \INA cells.



% ==  ==================================================================
% == Timelapse
% == ===================================================================

\subsubsection*{Single \INA Cells Detach Spontaneously from Aggregates of Critical Size}%
\label{C1:results_timelapse}%
Using time-lapse microscopy, we observed that \SIplusminus{26}{8}{\percent} of
\INA aggregates growing on single hMSCs spontaneously shed \INA cells
(\autoref{fig:2}A, B; Supplementary\,Video\,1). Notably, all detached cells
exhibited similar directional movements, suggesting entrainment in convective
streams generated by temperature gradients within the incubation chamber. \INA
predominantly detached from other \INA cells or aggregates (\autoref{fig:2}C),
indicating weaker adhesive forces in homotypic interactions than in heterotypic
interactions. The detachment frequency increased after \SI{52}{hours}, when most
aggregates that shed \INA cells were categorized as large (greater than 30
cells) (\autoref{fig:2}D). Since approximately 10-20 \INA cells already fully
covered a single hMSC, we suggest that myeloma cell detachment depended not only
on hMSC saturation but also required a minimum aggregate size. Interestingly,
\INA detached mostly as single cells, independent of aggregate size categories
\omnibus{F}{2,6}{4.68}{-unc}{0.059}
(Two-factor RM-ANOVA) (\autoref{fig:2}E),
showing that aggregates remained mostly stable despite losing cells.


\vspace{\vfull}

% == Fig. 2 ============================================================
\includeimage[1][{\fmleft} 7.43in {\fmright} {\fmtop}]{
    FIGS/figures9_2.pdf
}\figcaption[fig:2]{ %
    Time-lapse analysis of INA-6 detachment from INA-6 aggregates and hMSCs.
    \tile{A} Frequency of observed INA-6 aggregates that did or did not lose
    INA-6 cell(s). 87 aggregates were evaluated per datapoint. \tile{B} Example
    of a ``disseminating'' INA-6 aggregate growing on fluorescently (PKH26)
    stained hMSC (from A-D). Dashed green lines are trajectories of detached
    INA-6 cells. Scale bar = \SI{50}{\um}. \tile{\mbox{C-E}} Quantitative
    assessment of INA-6 detachments. 45 detachment events were evaluated per
    datapoint. Seeding ratio INA-6:MSC = 4:1. \tile{C} Most INA-6 cells
    dissociated from another INA-6 cell and not from an hMSC
    \omnibus{F}{1,3}{298}{-unc}{\scinot{4.2}{-4}}. \tile{D} Detachment frequency
    of aggregate size categories. \tile{E} Detachment frequency of INA-6 cells
    detaching as single, pairs or more than three cells. \tile{Statistics} (A):
    Paired-t-test; (C-E): Paired-t-test, Two-factor RM-ANOVA; Datapoints
    represent three (A) or four (C-E) independent time-lapse recordings of
    co-cultures with hMSCs from two (A) or three (C-E) unique donors.
}


\vspace{-.3cm}
\subsubsection*{Cell Division Generates a Daughter Cell Detached from hMSC}%
\label{C1:results_division}%
We suspected that cell division drives detachment because we observed that
MSC-adhering \INA cells could generate daughter cells that “roll over” the
mother cell (\autoref{fig:3}A; Supplementary\,Video\,2). We recorded and
categorized the movement of \INA daughter cells in confluent hMSCs after cell
division. Half of all \INA divisions yielded two daughter cells that remained
stationary, indicating hMSC adherence (\autoref{fig:3}B,\,C;
Supplementary\,Video\,3). The other half of division events generated one
hMSC-adhering cell and one non–hMSC-adhering cell, which
rolled around the \MAina cell for a median time of \SI{2.5}{hours} post division
(Q1=\SI{1.00}{hour}, Q3=\SI{6.25}{hours}) until it stopped and re-adhered to the
hMSC monolayer (\autoref{fig:3}D; Supplementary\,Video\,2,
Supplementary\,Video\,4). Thus, cell division establishes a time window in which
one daughter cell can detach.


% == Fig. 3 ============================================================
\begin{figure}
    \includeimage[1][{\fmleft} 5.43in {\fmright} {\fmtop}]{
        FIGS/figures9_3.pdf
    }\figcaption[fig:3]{ %
        Detachment of INA-6 daughter cells after Cell Division. \tile{A-D} INA-6
        divisions in interaction with confluent hMSCs. Seeding ratio INA-6:MSC =
        4:20. \tile{A} Three examples of dividing INA-6 cells generating either
        two MA, or one MA and one nMA daughter cells as described in (G). Dashed
        circles mark mother cells (white), MA cell (blue), and first position of
        nMA cell (green). Scale bar: \SI{20}{\um}. \tile{B} Cell division of
        \ac{MA} mother cell can yield one mobile \ac{nMA} daughter cell.
        \tile{C} Frequencies of INA-6 pairs defined in (A, B) per observed cell
        division. 65 divisions were evaluated for each of three independent
        time-lapse recordings. \tile{D} Rolling duration of nMA cells after
        division did not depend on hMSC donor \omnibus{H}{2}{5.250}{-unc}{.072}.
        Datapoints represent single nMA-cells after division. \tile{E-G}
        Adhesive and cell cycle assessment of MSC-interacting INA-6
        subpopulations using the V-Well assay. \tile{E} Schematic of V-Well
        Assay (see \apdxref{apdx:supplemental}{fig:S1} for detailed analysis).
        MSC-interacting subpopulations were separated by subsequent
        centrifugation and removal of the pellet. The pellet size was quantified
        by its total fluorescence brightness. Adhering subpopulations were
        resuspended by rough pipetting. \tile{F} Relative cell pellet sizes of
        adhesive INA-6 subpopulations that cycle either asynchronously or were
        synchronized at mitosis. Gray lines in-between points connect dependent
        measurements of co-cultures ($n=9$) that shared the same hMSC-donor and
        INA-6 culture. Co-cultures were incubated for three different durations
        (\SIlist{1;2;3}{\hour} after INA-6 addition). Time points were pooled,
        since time did not show an effect on cell adhesion
        \omnibus{F}{2,4}{1.414}{-unc}{0.343}
        Factorial RM-ANOVA shows an interaction between cell cycle and the kind
        of adhesive subpopulation \omnibus{F}{1,8}{42.67}{-unc}{\scinot{1.82}{-4}}.
        Technical replicates = 4 per datapoint. \tile{G} Cell cycles were profiled
        in cells gathered from the pellets of four independent co-cultures ($n=4$)
        and the frequency of G0/G1 cells are displayed depending on co-culture
        duration (see \apdxref{apdx:supplemental}{fig:S3} for cell cycle profiles).
        Four technical replicates were pooled after pelleting. \tile{Statistics}
        (D): Kruskal-Wallis H-test. (F): Paired t-test, (G): Paired t-test,
        two-factor RM-ANOVA. Datapoints represent INA-6 from independent co-
        cultures with hMSCs from three unique donors.
    }
\end{figure}

To validate that cell division reduced adhesion, we measured both the size and
cell cycle profile of the \nMAina and \MAina populations using an enhanced
V-well assay (method described in \autoref{fig:3}E,
\apdxref{apdx:supplemental}{fig:S1},\,\ref{fig:S2}). For comparison, we fully
synchronized and arrested \INA cells at mitosis and released their cell cycle
immediately before addition to the hMSC monolayer, rendering them more likely to
divide while adhering. Mitotic arrest significantly increased the number of
\nMAina cells and decreased the number of \MAina cells (\autoref{fig:3}F).
Furthermore, the \nMAina population contained significantly more cells cycling
in the G0/G1 phase than the \MAina population, both in synchronously and
asynchronously cycling \INA (\autoref{fig:3}G,
\apdxref{apdx:supplemental}{fig:S3},\,\ref{fig:S4}). The number of \nMAina \INA
cells increased due to a higher cell division frequency. Taken together, we
showed that \INA detach from aggregates by generating one temporarily detached
daughter cell after cell division, a process that potentially contributes to the
initiation of dissemination.



% ==  ==================================================================
% == RNAseq
% == ===================================================================

\subsubsection*{RNAseq of Non-MSC-Adhering and MSC-Adhering Subpopulations}%
\label{C1:results:RNAseq}%
%%%
To characterize the subpopulations separated by WPSC, we conducted RNAseq,
revealing 1291 differentially expressed genes between \versus{\nMAina}{\CMina},
484 between \versus{\MAina}{\CMina}, and 195 between \versus{\MAina}{\nMAina}. We
validated RNAseq and found that the differential expression of 18 genes
correlated with those measured with qPCR for each pairwise comparison
(\autoref{fig:4}C--E, \apdxref{apdx:supplemental}{fig:S5}):
\versus{\nMAina}{\CMina} \omnibus{\rho}{16}{.803}{}{\scinot{6.09}{-5}},
\versus{\MAina}{\CMina} \omnibus{\rho}{16}{.827}{}{\scinot{2.30}{-5}}, and
\versus{\MAina}{\nMAina} \omnibus{\rho}{16}{.746}{}{\scinot{3.74}{-4}}
(Spearman's rank correlation). One of the \SI{18}{} genes (\textit{MUC1})
measured by qPCR showed a mean expression opposite to that obtained by RNAseq
(\versus{\nMAina}{\CMina}), although the difference was insignificant
(\autoref{fig:4}C). For \versus{\nMAina}{\CMina}, the difference in expression
measured by qPCR was significant for only two of the 11 genes
(\listit{DKK1, OPG}), whereas the other genes (\listit{BCL6, BMP4, BTG2, IL10RB,
    IL24, NOTCH2, TNFRSF1A, TRAF5}) only confirmed the tendency measured by RNAseq
(\autoref{fig:4}C--E). For \versus{\MAina}{\CMina}, qPCR validated the
significant upregulation of seven genes (\listit{TGM2, DCN, LOX, MMP14, MMP2,
    CXCL12, CXCL8}), whereas the downregulation of \textit{BMP4} was insignificant.



% == Fig. 4 ============================================================
\includeimage[1][{\fmleft} 4.3in {\fmright} {\fmtop}]{
    FIGS/figures9_4.pdf
}\figcaption[fig:4]{ %
    Separation and gene expression of INA-6 subpopulations. \tile{A} Schematic
    of “Well-Plate Sandwich Centrifugation” (WPSC) separating nMA- from MA-INA6.
    A co-culture 96-well plate is turned upside down and attached on top of a
    “catching plate”, forming a “well-plate sandwich”. nMA-INA6 cells are
    collected in the catching plate by subsequent rounds of centrifugation and
    gentle washing. MA-INA6 are enzymatically dissociated from hMSCs or by rough
    pipetting. Subsequent RNAseq of MSC-interacting subpopulations reveals
    distinct expression clusters [right, multidimensional scaling plot (MDS)
            ($n=5$)]. \tile{B} Separation was microscopically tracked after each
    centrifugation step. \tile{C-E} RT-qPCR of genes derived from RNAseq
    results. Expression was normalized to the median of CM-INA6. Samples include
    those used for RNAseq and six further co-cultures ($n=11$; non-detects were
    discarded). \tile{C} Adhesion factors, ECM proteins, and matrix
    metalloproteinases. \tile{D} Factors involved in bone remodeling and bone
    homing chemokines. \tile{E} Factors involved in (immune) signaling.
    \tile{Statistics} (C-E): Paired t-test. Datapoints represent the mean of
    three (B-E) technical replicates. INA-6 were isolated from independent
    co-cultures with hMSCs from five (A, B), nine (C-E) unique donors.
}


\subsubsection*{Non-MSC-Adhering \INA and MSC-Adhering \INA Have Distinct Expression Patterns of Proliferation or Adhesion, Respectively}%
\label{C1:results:RNAseq:subpopulations}%
%%%
To functionally characterize the unique transcriptional patterns in \nMAina and
\MAina, we generated lists of genes that were differentially expressed
\textit{vs.} the other two subpopulations [termed \versus{nMA}{\MAandCM} and
        \versus{MA}{\nMAandCM}]. Functional enrichment analysis was performed, and the
enriched terms were displayed as ontology clusters (\autoref{fig:5}A). \nMAina
upregulated genes enriched with loosely connected term clusters associated with
proliferation (e.g., “positive regulation of cell cycle”). \MAina upregulated
genes enriched with tightly connected term clusters related to cell adhesion and
the production of ECM factors (e.g., ``cell-substrate adhesion''). Similar
ontology terms were enriched in the gene lists obtained from pairwise
comparisons \versus{nMA}{CM}, \versus{MA}{CM}, and \versus{MA}{nMA}
(\autoref{fig:5}B). In particular, \versus{nMA}{CM} (but not \versus{MA}{CM})
upregulated genes that were enriched with “G1/S transition”, showing that WPSC
isolated \nMAina\ daughter cells after cell division.

To check for similarities between lists of differentially expressed genes from
hMSC-interacting subpopulations, we performed enrichment analysis on gene lists
from the overlaps (“$\cap$”) between all pairwise comparisons (\autoref{fig:5}B,
\apdxref{apdx:supplemental}{fig:S6}), and showed the extent of these overlaps in
circos plots (\autoref{fig:5}C). The overlap between \versus{MA}{CM} and
\versus{nMA}{CM} showed neither enrichment with proliferation- nor
adhesion-related terms but with apoptosis-related terms. A direct comparison of
MSC-interacting subpopulations (\versus{MA}{nMA}) showed a major overlap with
\versus{MA}{CM} (\autoref{fig:5}C, middle). This overlap was enriched with terms
related to adhesion but not proliferation. Hence, \MAina\ and \nMAina\ mostly
differed in their expression of adhesion genes.

To assess whether \nMAina\ and \MAina\ were regulated by separate transcription
factors, we examined the enrichment of curated regulatory networks from the
TRRUST database (\autoref{fig:5}B, bottom). All the lists were enriched for p53
regulation. E2F1 regulation was observed only in genes upregulated in
\versus{nMA}{CM} and downregulated in \versus{MA}{nMA}. Genelists involving
\MAina\ were enriched in regulation by subunits of NF-$\kappa$B (NFKB1/p105 and
RELA/p65) and factors of immediate early response (SRF, JUN). Correspondingly,
NF-$\kappa$B and JUN are known to regulate the expression of adhesion factors in
multiple myeloma and B-cell lymphoma, respectively
\cite{blonskaJunregulatedGenesPromote2015,taiRoleBcellactivatingFactor2006}.

Taken together, MSC-interacting subpopulations showed unique regulatory
patterns, focusing on either proliferation or adhesion.


% == Fig. 5 ============================================================
\newpage
\vspace*{-1.4cm}
\includeimage[.95][{\fmleft} {\fmbottom} {\fmright} {\fmtop}]{
    FIGS/figures9_5.pdf
}\figcaption[fig:5]{ %
    Functional analysis of MSC-interacting subpopulations \tile{(A-C)}
    Functional enrichment analysis of differentially expressed genes (from
    RNAseq) using Metascape. \tile{A} Gene ontology (GO) cluster analysis of
    gene lists that are unique for MA (left) or nMA (right) INA-6. Circle nodes
    represent subsets of input genes falling into similar GO-term. Node size
    grows with the number of input genes. Node color defines a shared parent
    GO-term. Two nodes with a $\text{similarity score} > 0.3$ are linked.
    \tile{B} Enrichment analysis of pairwise comparisons between MA
    subpopulations and their overlaps (arranged in columns). GO terms were
    manually picked and categorized (arranged in rows). Raw Metascape results
    are shown in \apdxref{apdx:supplemental}{fig:S6}. For each GO-term, the
    p-values (x-axis) and the counts of matching input genes (circle size) were
    plotted. The lowest row shows enrichment of gene lists from the
    TRRUST-database. \tile{C} Circos plots by Metascape. Sections of a circle
    represent lists of differentially expressed genes. Purple lines connect same
    genes appearing in two gene lists. \(\cap\): Overlapping groups, MA:
    MSC-adhering, nMA: non-MSC-adhering, CM: MSC-Conditioned Medium. \tile{D}
    INA-6 were co-cultured on confluent hMSC for \SI{24}{\hour} or
    \SI{48}{\hour}, separated by WPSC and sub-cultured for \SI{48}{\hour} under
    IL-6 withdrawal ($n=6$), except the control (IL-6 + INA-6) ($n=3$). Signals
    were normalized (red line) to INA-6 cells grown without hMSCs and IL-6
    ($n=3$). \tile{Statistics} (D): Paired t-test, two-factor RM-ANOVA.
    Datapoints represent the mean of four technical replicates. INA-6 were
    isolated from independent co-cultures with hMSCs from six unique donors. }

\needspace{.1cm}

% == ===================================================================
% == Special Functional Characterization
% == ===================================================================


\subsubsection*{nMA-INA6 and MA-INA6 Show Increased Apoptosis Signaling Mediated by ER-Stress, p53 and Death Domain Receptors}%
\label{C1:results:RNAseq:apoptosis}%
%%%
As previously stated, apoptosis rates increased in \INA cells grown on confluent
hMSCs compared with \ac{CMina}\ cells after \SI{24}{hours} of co-culture
(\autoref{fig:1}D). Since this setup was similar to that used to separate
hMSC-interacting subpopulations using WPSC, we looked for enrichment of
apoptosis-related terms (\autoref{fig:5}B). “Regulation of cellular response to
stress” and “intrinsic apoptotic signaling pathway (in response to ER-stress)”
are terms that were enriched in \versus{nMA}{CM}, \versus{MA}{CM} and their
overlap. We also found specific stressors for either \nMAina\ (“intrinsic
apoptotic signaling pathway by p53 class mediator”) or \MAina\ (“extrinsic
apoptotic signaling pathway via death domain receptor”). Therefore, apoptosis
may be driven by ER stress in both \nMAina\ and \MAina, but also by individual
pathways such as p53 and death domain receptors, respectively.


\subsubsection*{nMA-INA6 and MA-INA6 Regulate Genes Associated with Bone Loss}%
\label{C1:results:RNAseq:bone_loss}%
%%%
Myeloma cells cause bone loss by degradation and dysregulation of bone turnover
via \textit{DKK1} and \textit{OPG}
\cite{standalOsteoprotegerinBoundInternalized2002,vanvalckenborghMultifunctionalRoleMatrix2004,zhouDickkopf1KeyRegulator2013}.
RNAseq of hMSC-interacting subpopulations showed enrichment with functional
terms ``skeletal system development'' and ``ossification'' (\autoref{fig:5}A,
\apdxref{apdx:supplemental}{fig:S6}), as well as the regulation of
\textit{MMP2}, \textit{MMP14}, \textit{DKK1}, and \textit{OPG}. Validation by
qPCR (\autoref{fig:4}C, D) showed that \MAina\ significantly upregulated both
\textit{MMP14} and \textit{MMP2} compared with either \nMAina\ or \CMina. The
expression of \textit{DKK1}, however, was upregulated significantly in \nMAina\
(and not significantly upregulated in \MAina), while \textit{OPG} was
significantly downregulated only in \nMAina.

Together, hMSC-interacting subpopulations might contribute to bone loss through
different mechanisms: \MAina\ expression of matrix metalloproteinases and
\nMAina\ cells via paracrine signaling.


\subsubsection*{MA-INA6 Upregulate Collagen and Chemokines Associated with Bone Marrow Retention}%
\label{C1:results:RNAseq:bone_retention}%
%%%
Retention of myeloma cells within the bone marrow is mediated by adhesion to the
ECM (e.g., collagen VI) and the secretion of chemokines (\textit{CXCL8} and
\textit{CXCL12}), potentially counteracting dissemination \cite{alsayedMechanismsRegulationCXCR42007,
    katzAdhesionMoleculesLifelines2010}.
RNAseq of hMSC-interacting subpopulations showed that genes upregulated in
\MAina\ were enriched with collagen biosynthesis and modifying enzymes, as well
as chemotaxis and chemotaxis-related terms (\autoref{fig:5}B). Using qPCR, we
validated the upregulation of collagen crosslinkers (\textit{LOX} and
\textit{TGM2}), collagen-binding \textit{DCN}, and chemokines (\textit{CXCL8}
and \textit{CXCL12}) in \MAina\ compared with both \nMAina\ and \CMina\
(\autoref{fig:4}D). Therefore, \MAina\ can provide both an adhesive surface and
soluble signals for the retention of malignant plasma cells in the bone marrow.


\subsubsection*{nMA-INA6 Show Highest Viability During IL-6 Withdrawal}%
\label{C1:results:RNAseq:viability}%
%%%
Although RNAseq did not reveal IL-6 induction in any WPSC-isolated
subpopulation, \nMAina\ upregulated \textit{IGF-1} 1.35-fold [RNAseq,
\versus{nMA}{\MAandCM}], which was shown to stimulate growth in CD45+ and IL-6
dependent myeloma cell lines such as INA-6, implying increased autonomy for
\nMAina\ (40). To test the autonomy of hMSC-interacting \INA subpopulations, we
isolated them using WPSC after \SI{24}{hours} and \SI{48}{hours} of co-culture,
sub-cultured them for \SI{48}{hours} under IL-6 withdrawal, and measured both
viability and apoptosis (\autoref{fig:5}D). Among the subpopulations, \nMAina\
was the most viable. Compared to \MAina, \nMAina\ increased cell viability by 8
or 4 fold when co-cultured for \SI{24}{hours} or \SI{48}{hours}, respectively
    [Hedges $g$ of Log\textsubscript{10}(Fold Change) = 2.31 or 0.82]. However, the
difference was no longer significant after \SI{48}{hours} of co-culture,
probably because \nMAina\ adhered to the hMSC layer (turning into \MAina) during
prolonged co-culture, which could also explain why the viability of \MAina\ cell
subcultures increased with prolonged co-culture. Nevertheless, \nMAina\ did not
achieve the same viability as that of \INA cells cultured with IL-6. Despite the
differences in viability, subcultures of hMSC-interacting subpopulations did not
show any differences in caspase 3/7 activity when co-cultured for \SI{48}{hours}
(\autoref{fig:5}D, right).

Overall, among the hMSC-interacting subpopulations, \nMAina\ had the highest
chance of surviving IL-6 withdrawal.


% == ===================================================================
% == Patient Survival
% == ===================================================================


% == Fig. 6 ============================================================
\newpage
\vspace*{-1.3cm}
\includeimage[1][{\fmleft} 3.4in {\fmright} {\fmtop}]{ FIGS/figures9_6.pdf }
\splitcaption[fig:6]{%     
    \nopagebreak%
    Survival of patients with multiple myeloma regarding the expression levels
    of adhesion and bone retention genes. \tile{A} p-value distribution of genes
    associated with patient survival ($n=535$) depending on high or low
    expression levels. Red dashed line marks the significance threshold of
    $p\text{-adj}=0.05$. Histogram of $p$-values was plotted using a bin width
    of $-\log_{10}(0.05)/2$. Patients with high and low gene expression were
    delineated using maximally selected rank statistics (maxstat). \tile{B}
    Survival curves for three genes taken from the list of adhesion genes shown
    in (A), maxstat thresholds defining high and low expression were:
    \textit{CXCL12}: 81.08; \textit{DCN}: 0.75; \textit{TGM2}:
    \SI{0.66}{\normcounts}.
}{%
    \tile{C} Gene expression (RNAseq, $n=873$) measured
    in normalized counts (\texttt{edgeR}) of \textit{CXCL12}, \textit{DCN} in Bone Marrow
    Plasma Cell (BMPC), Monoclonal Gammopathy of Undetermined Significance
    (MGUS), smoldering Multiple Myeloma (sMM), Multiple Myeloma (MM), Multiple
    Myeloma Relapse (MMR), Human Myeloma Cell Lines (HMCL). The red dashed line
    marks one normalized read count. \tile{Statistics} (A, B): Log-rank test;
    (C): Kruskal-Wallis, Mann–Whitney U Test. All $p$-values were corrected
    using the Benjamini-Hochberg procedure.
}

\subsubsection*{Genes Upregulated by MA-INA6 are Associated with an Improved Disease Prognosis}%
\label{C1:results:RNAseq:patient_survival}%
%%%
To relate the adhesion of \MAina\ observed \textit{in vitro} to the progression
of multiple myeloma, we assessed patient survival [$n = 535$,
        \citet{seckingerTargetExpressionGeneration2017b,seckingerCD38ImmunotherapeuticTarget2018}]
depending on the expression level of 101 genes, which were upregulated in
\versus{MA}{\nMAandCM} and are part of the ontology terms ``Extracellular matrix
organization,'' ``ECM proteoglycans,'' ``cell-substrate adhesion,'' and
``negative regulation of cell-substrate adhesion'' (\autoref{fig:6}A,
\apdxref{apdx:supplemental}{tab:S2}). As a reference, we generated a list of 173
cell cycle-related genes that were upregulated by \versus{nMA}{\MAandCM}.

As expected, longer patient survival was associated with low expression of the
majority of cell cycle genes [71 or 68 genes for \ac{PFS}
        or \ac{OS}]. Only a few cell cycle genes (two for PFS and seven
for OS) were associated with survival when highly expressed. Intriguingly,
adhesion genes showed an inverse pattern: a large group of adhesion genes (24
for PFS and 26 for OS) was significantly associated with improved survival when
highly expressed, whereas only a few genes (two for PFS and four for OS)
improved survival when expressed at low levels (\autoref{tab:1}). We concluded
that the myeloma-dependent expression of adhesion factors determined in our
\textit{in vitro} study correlates with improved patient survival.


\subsubsection*{Expression of Adhesion- or Retention-related Genes (\textit{CXCL12}, \textit{DCN} and \textit{TGM2}) is Decreased During Progression of Multiple Myeloma}%
\label{C1:results:RNAseq:progression}%
%%%
To examine how the disease stage affects the adhesion and bone marrow retention
of myeloma cells \textit{in vitro}, we analyzed the expression of
\textit{CXCL12} in healthy plasma cell (BMPC) cohorts of patients at different
disease stages and in myeloma cell lines (HMCL)
[described in \citet{seckingerCD38ImmunotherapeuticTarget2018}] (\autoref{fig:6}C). We also
included \textit{DCN} and \textit{TGM2} since both are suggested to inhibit
metastasis in different cancers by promoting cell-matrix interactions
\cite{huDecorinmediatedSuppressionTumorigenesis2021,tabolacciRoleTissueTransglutaminase2019}.
In accordance with independent reports
\cite{huangHigherDecorinLevels2015,baoCXCR4GoodSurvival2013}, high
expression of \textit{CXCL12} and \textit{DCN} by myeloma cells was associated
with improved overall survival (adj. \( p = .009 \) and \( .008 \),
respectively) (\autoref{fig:6}B).




    % == Tab. 1 ============================================================
    {
        \footnotesize
        \def\myheader{
            \hline
            \textbf{Regulation during disease progression} & \textbf{Gene} & \textbf{Ensemble ID} & \textbf{Progression Free / Overall Survival} & \textbf{Better Prognosis with high/low expression} & \multicolumn{2}{p{3cm}|}{\textbf{Association of expression with survival}}                    \\
            \hhline{~~~~~--}
            &               &                      &                                              &                                                    & \textbf{[p-unc]}                                                      & \textbf{[p-adj]} \\
            \hline
        }

        \begin{longtable}{|>{\bfseries}p{3cm}|>{\bfseries}p{1.9cm}|p{3cm}|p{2cm}|p{2cm}|p{1.5cm}|p{1.5cm}|}
            \caption{Adhesion and ECM genes (shown in \autoref{fig:6}A) were
                filtered by their association with patient survival (p-adj. < 0.01)
                and were categorized as continuously downregulated during disease
                progression. The complete list is presented in
                \apdxref{apdx:supplemental}{tab:S2}. Bone Marrow Plasma Cells
                (BMPC), Monoclonal Gammopathy of Undetermined Significance (MGUS),
                smoldering Multiple Myeloma (sMM), Multiple Myeloma (MM), and
                Multiple Myeloma Relapse (MMR). p-unc: unadjusted p-values; p-adj:
                p-values adjusted using the Benjamini-Hochberg method with 101
            genes.}\label{tab:1}                                                  \\
            \myheader
            \endfirsthead

            \longtablecaptions{3}{}

            \hline

            \multirowcell{3}{3cm}{Not Downregulated (or overall low expression)}
             & CCNE2  & ENSG00000175305 & Overall    & low  & 5.34E-04 & 8.64E-03 \\
            \hhline{~======}
             & MMP2   & ENSG00000087245 & Prog. Free & high & 2.29E-05 & 2.32E-03 \\
            \hhline{~======}
             & OSMR   & ENSG00000145623 & Prog. Free & high & 5.67E-04 & 7.15E-03 \\
            \hhline{~======}
            \hline

            \multirowcell{8}{2.7cm}{Continuously Downregulated (BMPC > MGUS > sMM > MM > MMR)}
             & AXL    & ENSG00000167601 & Overall    & high & 3.64E-05 & 1.84E-03 \\
            \hhline{~======}
             & COL1A1 & ENSG00000108821 & Prog. Free & high & 3.03E-04 & 4.37E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 5.93E-04 & 8.64E-03 \\
            \hhline{~======}
             & CXCL12 & ENSG00000107562 & Prog. Free & high & 1.16E-04 & 2.93E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 6.48E-04 & 8.64E-03 \\
            \hhline{~======}
             & CYP1B1 & ENSG00000138061 & Overall    & high & 6.84E-04 & 8.64E-03 \\
            \hhline{~======}
             & DCN    & ENSG00000011465 & Overall    & high & 2.47E-04 & 8.33E-03 \\
            \hhline{~======}
             & LRP1   & ENSG00000123384 & Overall    & high & 4.34E-04 & 8.64E-03 \\
            \hhline{~======}
             & LTBP2  & ENSG00000119681 & Prog. Free & high & 9.03E-05 & 2.93E-03 \\
            \hhline{~======}
             & CYP1B1 & ENSG00000138061 & Overall    & high & 6.84E-04 & 8.64E-03 \\
            \hhline{~======}
             & DCN    & ENSG00000011465 & Overall    & high & 2.47E-04 & 8.33E-03 \\
            \hhline{~======}
             & LRP1   & ENSG00000123384 & Overall    & high & 4.34E-04 & 8.64E-03 \\
            \hhline{~======}
             & LTBP2  & ENSG00000119681 & Prog. Free & high & 9.03E-05 & 2.93E-03 \\
            \hhline{~======}
             & MFAP5  & ENSG00000197614 & Prog. Free & high & 2.43E-04 & 4.09E-03 \\
            \hhline{~======}
             & MMP14  & ENSG00000157227 & Prog. Free & high & 6.93E-05 & 2.93E-03 \\
            \hhline{~======}
             & MYL9   & ENSG00000101335 & Prog. Free & high & 1.46E-04 & 2.95E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 1.56E-05 & 1.57E-03 \\
            \hline
        \end{longtable}
    }


\textit{CXCL12} is expressed by BMPCs (median = 219~normalized counts), but its
expression levels are significantly lower from MGUS to relapsed multiple myeloma
(MMR) (median = 9~normalized counts in MMR and absent expression in most HMCL).
\textit{DCN} (but not \textit{TGM2}) was weakly expressed in BMPCs ($Q_1$ = 0.7,
$Q_3$ = 3.7, normalized counts), whereas \textit{TGM2} was weakly expressed only
in patients with monoclonal gammopathy of undetermined significance (MGUS)
($Q_1$ = 0.4, $Q_3$ = 4.1 normalized counts). The median and upper quartiles of
both \textit{DCN}- and \textit{TGM2} decreased continuously after each stage,
ending at $Q_3$ = 0.9 and $Q_3$ = 0.6, respectively, in MMR. 49 of the 101 adhesion genes
(\autoref{fig:6}A) followed a similar pattern of continuous downregulation in
the advanced stages of multiple myeloma (\apdxref{apdx:supplemental}{fig:S7}\,and\,\ref{fig:S8}), of which 19 genes were associated with
longer PFS when they were highly expressed. The other 52 (out of 101) adhesion
genes that were not downregulated across disease progression (or were expressed
at a level too low to make that categorization) contained only five genes that
were associated with longer PFS at high expression (\autoref{tab:1},
\apdxref{apdx:supplemental}{tab:S2}).

Together, the expression of adhesion or bone marrow retention-related markers
(\textit{CXCL12}, \textit{DCN}, and \textit{TGM2}) is reduced or lost at
advanced stages of multiple myeloma, which could enhance dissemination and
reduce retention in the BM microenvironment.


\newpage

% ======================================================================
% ======================================================================
% ======================================================================
% == DISCUSSION

\unnsubsection{Discussion}\label{C1:discussion} % > Require unique label
%%%
In this study, we developed an \textit{in vitro} model to investigate the
attachment/detachment dynamics of \INA\ cells to/from hMSCs and established
methods to isolate the attached and detached intermediates \nMAina\ and \MAina.
Secondly, we characterized a cycle of (re)attachment, division, and detachment,
linking cell division to the switch that causes myeloma cells to detach from
hMSC adhesion (\autoref{fig:7}). Thirdly, we identified clinically relevant
genes associated with patient survival, where better or worse survival was based
on the adherence status of \INA\ to hMSCs.

\INA\ cells emerged as a robust choice for studying myeloma dissemination
\textit{in vitro}, showing rapid and strong adherence, as well as aggregation
exceeding MSC saturation. The IL-6 dependency of \INA\ enhanced the resemblance
of myeloma cell lines to patient samples, with \INA\ ranking 13th among 66 cell
lines \cite{sarinEvaluatingEfficacyMultiple2020}. Despite variations in bone
marrow MSCs between multiple myeloma and healthy states, we anticipated the
robustness of our results, given the persistent strong adherence and growth
signaling from MSCs to \INA\ during co-cultures
\cite{dotterweichContactMyelomaCells2016}.


We acknowledge that \INA\ cells alone cannot fully represent the complexity of
myeloma aggregation and detachment dynamics. However, the diverse adhesive
properties of myeloma cell lines pose a challenge. We reasoned that attempting
to capture this complexity within a single publication would not be possible.
Our focus on \INA\ interactions with hMSCs allowed for a detailed exploration of
the observed phenomena, such as the unique aggregation capabilities that
facilitate the easy detection of detaching cells \textit{in vitro}. The validity
of our data was demonstrated by matching the \textit{in vitro} findings with the
gene expression and survival data of the patients (e.g., \textit{CXCL12},
\textit{DCN}, and \textit{TGM2} expression, \( n=873 \)), ensuring biological
consistency and generalizability regardless of the cell line used. The protocols
presented in this study offer a cost-efficient and convenient solution, making
them potentially valuable for a broader study of cell interactions. We encourage
optimizations to meet the varied adhesive properties of the samples, such as
decreasing the number of washing steps if the adhesive strength is low. We
caution against strategies that average over multiple cell lines without prior
understanding their diverse attachment/detachment dynamics, such as homotypic
aggregation. Such detailed insights may prove instrumental when considering the
diversity of myeloma patient samples across different disease stages
\cite{kawanoHomotypicCellAggregations1991a, okunoVitroGrowthPattern1991}.

The intermediates, \nMAina\ and \MAina, were distinct but shared similarities in
response to cell stress, intrinsic apoptosis, and regulation by p53. Unique
regulatory patterns were related to central transcription factors: E2F1 for
\nMAina; and NF-$\kappa$B, SRF, and JUN for \MAina. This distinction may have
been established through antagonism between p53 and the NF-$\kappa$B subunit
RELA/p65 \cite{wadgaonkarCREBbindingProteinNuclear1999, websterTranscriptionalCrossTalk1999}. Similar
regulatory patterns were found in transwell experiments with RPMI1-8226 myeloma
cells, where direct contact with the MSC cell line HS5 led to NF-$\kappa$B
signaling and soluble factors to E2F signaling \cite{dziadowiczBoneMarrowStromaInduced2022}.


% == Fig. 7 ============================================================
\includeimage[1][{\fmleft} 6.8in {\fmright} {\fmtop}]{
    FIGS/figures9_7.pdf
}\figcaption[fig:7]{
    Proposed model of “Detached Daughter Driven Dissemination” (DDDD) in
    aggregating multiple myeloma. \tile{Heterotypic Interaction} Malignant
    plasma cells colonize the bone marrow microenvironment by adhering to an MSC
    (or osteoblast, ECM, etc.) to maximize growth and survival through paracrine
    and adhesion mediated signaling, even if contact may trigger initial
    apoptosis. Gene expression will focus on establishing a strong anchor within
    the bone marrow, but also on attracting other myeloma cells (via secretion
    of ECM factors and CXCL12/CXCL8, respectively). \tile{Cell Division} Cell
    fission can generate one daughter cell that no longer adheres to the MSC
    (nMA). \tile{Homotypic Interaction} If myeloma cells have the capacity to
    grow as aggregates, the daughter cell stays attached to their MSC-adhering
    mother cell (MA). \tile{Re-Adhesion} The daughter cell “rolls around” the
    mother cell until it re-adheres to the MSC. Our model estimates the rolling
    duration to be \SIrange{1}{10}{\hour} long. \tile{Proliferation \& Saturation} We
    estimate that a single myeloma cell covers one MSC completely after roughly
    four population doublings. When heterotypic adhesion is saturated,
    subsequent daughter cells benefit from a homotypic interaction, since they
    stay close to growth-factor secreting MSCs and focus gene expression on
    proliferation (e.g. driven by E2F) and not adhesion (driven by NF-$\kappa$B).
    \tile{Critical Size} Homotypic interaction is weaker than heterotypic
    interaction, and each cell fission destabilizes the aggregate. Hence,
    detachment of myeloma cells may depend mostly on aggregate size.
    \tile{Dissemination} After myeloma cells have detached, they gained a
    viability advantage through IL-6-independence (with unknown duration), which
    enhances their survival outside of the bone marrow and allows them to spread
    throughout the body.
}



The first subpopulation, \nMAina, represented proliferative and disseminative
cells; They drove detachment through cell division, which was regulated by E2F,
p53, and likely their crosstalk \cite{polagerP53E2fPartners2009}. \nMAina
upregulate cell cycle progression genes associated with worse prognosis, because
proliferation is a general risk factor for an aggressive disease course
\cite{hoseProliferationCentralIndependent2011}. Additionally, \nMAina\ survived
IL-6 withdrawal better than \CMina\ and \MAina, implying their ability to
proliferate independently of the bone marrow
\cite{bladeExtramedullaryDiseaseMultiple2022}. Indeed, xenografted \INA\ cells
developed autocrine IL-6 signaling but remained IL-6-dependent after
explantation \cite{burgerGp130RasMediated2001c}. The increased autonomy of
\nMAina\ cells can be explained by the upregulation of \textit{IGF-1}, being the
major growth factor for myeloma cell lines \cite{sprynskiRoleIGF1Major2009}.
Other reports characterized disseminating cells differently: Unlike \nMAina,
circulating myeloma tumor cells were reported to be non-proliferative and bone
marrow retentive \cite{garcesTranscriptionalProfilingCirculating2020}. In
contrast to circulating myeloma tumor cells, \nMAina\ were isolated shortly
after detachment and therefore these cells are not representative of further
steps of dissemination, such as intravasation, circulation or intravascular
arrest \cite{zeissigTumourDisseminationMultiple2020}. Furthermore, Brandl et al.
described proliferative and disseminative myeloma cells as separate entities,
depending on the surface expression of CD138 or JAM-C
\cite{akhmetzyanovaDynamicCD138Surface2020,
    brandlJunctionalAdhesionMolecule2022}. Although CD138 was not differentially
regulated in \nMAina\ or \MAina, both subpopulations upregulated JAM-C,
indicating disease progression \cite{brandlJunctionalAdhesionMolecule2022}.

Furthermore, \nMAina\ showed that cell division directly contributed to
dissemination. This was because \INA\ daughter cells emerged from the mother
cell with distance to the hMSC plane in the 2D setup. A similar mechanism was
described in an intravasation model in which tumor cells disrupt the vessel
endothelium through cell division and detach into blood circulation
\cite{wongMitosismediatedIntravasationTissueengineered2017}. Overall, cell
division offers key mechanistic insights into dissemination and metastasis.



The other subpopulation, \MAina, represented cells retained in the bone marrow; \linebreak
\MAina\ strongly adhered to MSCs, showed NF-$\kappa$B signaling, and upregulated
several retention, adhesion, and ECM factors. The production of ECM-associated
factors has recently been described in MM.1S and RPMI-8226 myeloma cells
\cite{maichlIdentificationNOTCHdrivenMatrisomeassociated2023}. Another report did not
identify the upregulation of such factors after direct contact with the MSC cell
line HS5; hence, primary hMSCs may be crucial for studying myeloma-MSC
interactions \cite{dziadowiczBoneMarrowStromaInduced2022}. Moreover, \MAina\ upregulated
adhesion genes associated with prolonged patient survival and showed decreased
expression in relapsed myeloma. As myeloma progression implies the independence
of myeloma cells from the bone marrow \cite{bladeExtramedullaryDiseaseMultiple2022,sarinEvaluatingEfficacyMultiple2020}, we interpreted these adhesion genes as
mediators of bone marrow retention, decreasing the risk for dissemination and
thereby potentially prolonging patient survival. However, the overall impact of
cell adhesion and ECM on patient survival remains unclear. Several adhesion
factors have been proposed as potential therapeutic targets
\cite{brandlJunctionalAdhesionMolecule2022, bouzerdanAdhesionMoleculesMultiple2022}. Recent studies
have described the prognostic value of multiple ECM genes, such as those driven
by NOTCH \cite{maichlIdentificationNOTCHdrivenMatrisomeassociated2023}. Another study focused
on ECM gene families, of which only six of the 26 genes overlapped with our gene
set (\apdxref{apdx:supplemental}{tab:S2}) \cite{eversPrognosticValueExtracellular2023}. The
expression of only one gene (\textit{COL4A1}) showed a different association with overall
survival than that in our cohort. The lack of overlap and differences can be
explained by dissimilar definitions of gene sets (homology \textit{vs.} gene ontology),
methodological discrepancies, and cohort composition.

In summary, our \textit{in vitro} model provides a starting point for
understanding the initiation of dissemination and its implications for patient
survival, providing innovative methods, mechanistic insights into
attachment/detachment, and a set of clinically relevant genes that play a role
in bone marrow retention. These results and methods might prove useful when
facing the heterogeneity of disseminative behaviors among myeloma cell lines and
primary materials.



