



% ======================================================================
% == Chapter 1
% ======================================================================
\unnsection{Chapter 1: Modelling Myeloma Dissemination \textit{in vitro}}
\vspace{-\baselineskip} % > Remove space made by empty lines

% ## Reset reference counters of figs, tabs, so each chapter starts at 1
\setcounter{figure}{0}
\setcounter{table}{0}

% Title: Keep it Together: Modelling Myeloma Dissemination in vitro with
% hMSC- Interacting Subpopulations of INA-6 Cells and their
% Aggregation/Detachment Dynamics 

% ======================================================================
% == Abstract
% ======================================================================



% > First argument is the label
% > Second argument is the header name
% > Third argument is the abstract text
\customabstract{c1:abstract}{Abstract}{
    lorem ipsum
}


% ======================================================================
% == Introduction
% ======================================================================
% \unnsubsection{C1:Introduction}{Introduction}
\unnsubsection{Introduction}\label{C1:introduction} % > Require unique label
\ %
lorem ipsum dolor sit amet


% ======================================================================
% == Methods
% ======================================================================
\unnsubsection{Methods}
\ %
Methods are described in \refapdx{subapdx:methods}



% ======================================================================
% == Results
% ======================================================================
\unnsubsection{Results}
\ %
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% ## Example:
% > \includeimage[scale][left bottom right top]{
% >     pathtopicture
% > }{caption}


Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam


% ## Fig. 1
% > use \noindent, for some reason first image is indented otherwise
\noindent\includeimage[1][{\fmleft} 2.8in {\fmright} {\fmtop}]{
    FIGS/figures9_1.pdf
}\splitcaption{ %
INA-6 growth conformations and survival on hMSCs. \tile{A} Interaction of INA-6
(green) with hMSCs (black, negative staining) at different INA-6 densities
(constant hMSC densities). \mbox{Scale bar = \SI{200}{\um}}. \tile{B} Frequency of
single hMSCs (same as A) that are covered by INA-6 of varying group sizes.
Technical replicates = three per datapoint; Single hMSCs evaluated: 100 per
technical replicate. \tile{C} Interaction of INA-6 with hMSCs at different hMSC
densities (constant INA-6 densities). Scale bar = \SI{300}{\um}. 
}{%
\tile{D} Two types of homotypic interaction: Attachment after cell contact and
sustained attachment of daughter cells after cell division. Datapoints represent
one of four independent time-lapse recordings, each evaluating 116 interaction
events. \tile{E} Effects of hMSC-density on the viability (ATP, top) and
apoptosis (Caspase3/7 activity, bottom). INA-6:MSC ratio = 4:1; \mbox{Technical
replicates = four per datapoint}; E left: Signals were measured in INA-6 washed
off from hMSCs and normalized by INA-6 cultured in MSC-conditioned medium ($=
\text{red line}$) ($n=4$). E right: Signals were measured in co-cultures and
normalized by the sum of the signals measured in hMSC and INA-6 cultured
separately ($= \text{red line}$) ($n=3$). \tile{Statistics} Paired t-test,
two-factor RM- ANOVA. Datapoints represent independent co-cultures with hMSCs
from three (A, B, D, E right), four (E left) unique donors. Confl. = Confluent.
}
\label{fig:1}



Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet


% ## Fig. 2
\includeimage[1][{\fmleft} 7.4in {\fmright} {\fmtop}]{
    FIGS/figures9_2.pdf
}\captionof{figure}{ %
    Time-lapse analysis of INA-6 detachment from INA-6 aggregates and hMSCs.
    \tile{A} Frequency of observed INA-6 aggregates that did or did not lose
    INA-6 cell(s). 87 aggregates were evaluated per datapoint. \tile{B} Example
    of a "disseminating" INA-6 aggregate growing on fluorescently (PKH26)
    stained hMSC (from A-D). Dashed green lines are trajectories of detached
    INA-6 cells. Scale bar = \SI{50}{\um}. \tile{\mbox{C-E}} Quantitative assessment of
    INA-6 detachments. 45 detachment events were evaluated per datapoint.
    Seeding ratio INA-6:MSC = 4:1. \tile{C}
    Most INA-6 cells dissociated from another INA-6 cell and not from an hMSC
        [$F(1, 3) = 298$, $p\text{-unc}=4.2e-4$]. \tile{D} Detachment frequency of
    aggregate size categories. \tile{E} Detachment frequency of INA-6 cells
    detaching as single, pairs or more than three cells. \tile{Statistics} (A):
    Paired-t-test; (C-E): Paired-t-test, Two-factor RM-ANOVA; Datapoints
    represent three (A) or four (C-E) independent time-lapse recordings of
    co-cultures with hMSCs from two (A) or three (C-E) unique donors.
}\label{fig:2}



Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet


% ## Fig. 3
\includeimage[1][{\fmleft} 5.4in {\fmright} {\fmtop}]{
    FIGS/figures9_3.pdf
}\captionof{figure}{ %
    Detachment of INA-6 daughter cells after Cell Division. \tile{A-D} INA-6
    divisions in interaction with confluent hMSCs. Seeding ratio INA-6:MSC = 4:20.
    \tile{A} Three examples of dividing INA-6 cells generating either two MA, or one
    MA and one nMA daughter cells as described in (G). Dashed circles mark mother
    cells (white), MA cell (blue), and first position of nMA cell (green). Scale
    bar: \SI{20}{\um}. \tile{B} Cell division of MSC-adhering (MA) mother cell can
    yield one mobile non-MSC-adhering (nMA) daughter cell. \tile{C} Frequencies of
    INA-6 pairs defined in (A, B) per observed cell division. 65 divisions were
    evaluated for each of three independent time-lapse recordings. \tile{D} Rolling
    duration of nMA cells after division did not depend on hMSC donor [$H(2) =
                5.250$, $p\text{-unc} = .072$]. Datapoints represent single nMA-cells after
    division. \tile{E-G} Adhesive and cell cycle assessment of MSC-interacting INA-6
    subpopulations using the V-Well assay. \tile{E} Schematic of V-Well Assay (see
    \refapdx{apdx:supplemental}{fig:S1} for detailed analysis). MSC-interacting
    subpopulations were separated by subsequent centrifugation and removal of the
    pellet. The pellet size was quantified by its total fluorescence brightness.
    Adhering subpopulations were resuspended by rough pipetting. \tile{F} Relative
    cell pellet sizes of adhesive INA-6 subpopulations that cycle either
    asynchronously or were synchronized at mitosis. Gray lines in-between points
    connect dependent measurements of co-cultures ($n=9$) that shared the same
    hMSC-donor and INA-6 culture. Co-cultures were incubated for three different
    durations (\SIlist{1;2;3}{\hour} after INA-6 addition). Time points were pooled,
    since time did not show an effect on cell adhesion [$F(2,4) = 1.414$,
            $p\text{-unc} = 0.343$]. Factorial RM-ANOVA shows an interaction between cell
    cycle and the kind of adhesive subpopulation [$F(1, 8) = 42.67$, $p\text{-unc} =
                1.82e-04$]. Technical replicates = 4 per datapoint. \tile{G} Cell cycles were
    profiled in cells gathered from the pellets of four independent co-cultures
    ($n=4$) and the frequency of G0/G1 cells are displayed depending on co-culture
    duration (see \refapdx{apdx:supplemental}{fig:S3} for cell cycle profiles). Four
    technical replicates were pooled after pelleting. \tile{Statistics} (D):
    Kruskal-Wallis H-test. (F): Paired t-test, (G): Paired t-test, two-factor
    RM-ANOVA. Datapoints represent INA-6 from independent co- cultures with hMSCs
    from three unique donors. }\label{fig:3}


Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% ## Fig. 4
\includeimage[1][{\fmleft} 4.3in {\fmright} {\fmtop}]{
    FIGS/figures9_4.pdf
}\captionof{figure}{ %
    Separation and gene expression of INA-6 subpopulations. \tile{A} Schematic of
    “Well-Plate Sandwich Centrifugation” (WPSC) separating nMA- from MA-INA6. A
    co-culture 96-well plate is turned upside down and attached on top of a
    “catching plate”, forming a “well-plate sandwich”. nMA-INA6 cells are collected
    in the catching plate by subsequent rounds of centrifugation and gentle washing.
    MA-INA6 are enzymatically dissociated from hMSCs or by rough pipetting.
    Subsequent RNAseq of MSC-interacting subpopulations reveals distinct expression
    clusters [right, multidimensional scaling plot (MDS) ($n=5$)]. \tile{B}
    Separation was microscopically tracked after each centrifugation step.
    \tile{C-E} RT-qPCR of genes derived from RNAseq results. Expression was
    normalized to the median of CM-INA6. Samples include those used for RNAseq and
    six further co-cultures ($n=11$; non-detects were discarded). \tile{C} Adhesion
    factors, ECM proteins, and matrix metalloproteinases. \tile{D} Factors involved
    in bone remodeling and bone homing chemokines. \tile{E} Factors involved in
    (immune) signaling. \tile{Statistics} (C-E): Paired t-test. Datapoints represent the
    mean of three (B-E) technical replicates. INA-6 were isolated from independent
    co-cultures with hMSCs from five (A, B), nine (C-E) unique donors.
}\label{fig:4} % > Is referenced in supplemental



Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam



% ## Fig. 5
\includeimage[.97][{\fmleft} {\fmbottom} {\fmright} {\fmtop}]{
    FIGS/figures9_5.pdf
}\captionof{figure}{ %
    Functional analysis of MSC-interacting subpopulations \tile{(A-C)}
    Functional enrichment analysis of differentially expressed genes (from
    RNAseq) using Metascape. \tile{A} Gene ontology (GO) cluster analysis of
    gene lists that are unique for MA (left) or nMA (right) INA-6. Circle nodes
    represent subsets of input genes falling into similar GO-term. Node size
    grows with the number of input genes. Node color defines a shared parent
    GO-term. Two nodes with a $\text{similarity score} > 0.3$ are linked. \tile{B}
    Enrichment analysis of pairwise comparisons between MA subpopulations and
    their overlaps (arranged in columns). GO terms were manually picked and
    categorized (arranged in rows). Raw Metascape results are shown in
    \refapdx{apdx:supplemental}{fig:S6}. For each GO-term, the p-values (x-axis) and the
    counts of matching input genes (circle size) were plotted. The lowest row
    shows enrichment of gene lists from the TRRUST-database. \tile{C} Circos
    plots by Metascape. Sections of a circle represent lists of differentially
    expressed genes. Purple lines connect same genes appearing in two gene
    lists. \(\cap\): Overlapping groups, MA: MSC-adhering, nMA:
    non-MSC-adhering, CM: MSC-Conditioned Medium. \tile{D} INA-6 were
    co-cultured on confluent hMSC for \SI{24}{\hour} or \SI{48}{\hour},
    separated by WPSC and sub-cultured for \SI{48}{\hour} under IL-6 withdrawal
    ($n=6$), except the control (IL-6 + INA-6) ($n=3$). Signals were
    normalized (red line) to INA-6 cells grown without hMSCs and IL-6 ($n=3$).
    \tile{Statistics} (D): Paired t-test, two-factor RM-ANOVA. Datapoints represent the
    mean of four technical replicates. INA-6 were isolated from independent
    co-cultures with hMSCs from six unique donors. }\label{fig:5}


Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% ## Fig. 6
\includeimage[1][{\fmleft} 3.3in {\fmright} {\fmtop}]{ FIGS/figures9_6.pdf }
\splitcaption{%     
    Survival of patients with multiple myeloma regarding the expression levels
    of adhesion and bone retention genes. \tile{A} p-value distribution of genes
    associated with patient survival ($n=535$) depending on high or low
    expression levels. Red dashed line marks the significance threshold of
    $p\text{-adj}=0.05$. Histogram of $p$-values was plotted using a bin width
    of $-\log_{10}(0.05)/2$. Patients with high and low gene expression were
    delineated using maximally selected rank statistics (maxstat). \tile{B}
    Survival curves for three genes taken from the list of adhesion genes shown
    in (A), maxstat thresholds defining high and low expression were:
    \textit{CXCL12}: 81.08; \textit{DCN}: 0.75; \textit{TGM2}:
    \SI{0.66}{\normcounts}.
    \tile{C} Gene expression (RNAseq, $n=873$) measured
    in normalized counts (edgeR) of \textit{CXCL12}, \textit{DCN} in Bone Marrow
    Plasma Cell (BMPC), Monoclonal Gammopathy of Undetermined Significance
    (MGUS), smoldering Multiple Myeloma (sMM), Multiple Myeloma (MM), Multiple
}{%
    Myeloma Relapse (MMR), Human Myeloma Cell Lines (HMCL). The red dashed line
    marks one normalized read count. \tile{Statistics} (A, B): Log-rank test;
    (C): Kruskal-Wallis, Mann–Whitney U Test. All $p$-values were corrected
    using the Benjamini-Hochberg procedure.
}\label{fig:6}




Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

    % ## Tab. 1
    {
        \footnotesize
        \newcommand{\myheader}{
            \hline
            \textbf{Regulation during disease progression} & \textbf{Gene} & \textbf{Ensemble ID} & \textbf{Progression Free / Overall Survival} & \textbf{Better Prognosis with high/low expression} & \multicolumn{2}{p{3cm}|}{\textbf{Association of expression with survival}}                    \\
            \hhline{~~~~~--}
            &               &                      &                                              &                                                    & \textbf{[p-unc]}                                                      & \textbf{[p-adj]} \\
            \hline
        }

        \begin{longtable}{|>{\bfseries}p{3cm}|>{\bfseries}p{1.9cm}|p{3cm}|p{2cm}|p{2cm}|p{1.5cm}|p{1.5cm}|}
            \caption{Adhesion and ECM genes (shown in \autoref{fig:6}A) were
                filtered by their association with patient survival (p-adj. < 0.01)
                and were categorized as continuously downregulated during disease
                progression. The complete list is presented in
                \refapdx{apdx:supplemental}{tab:S2}. Bone Marrow Plasma Cells
                (BMPC), Monoclonal Gammopathy of Undetermined Significance (MGUS),
                smoldering Multiple Myeloma (sMM), Multiple Myeloma (MM), and
                Multiple Myeloma Relapse (MMR). p-unc: unadjusted p-values; p-adj:
                p-values adjusted using the Benjamini-Hochberg method with 101
            genes.}\label{tab:1}                                                  \\
            \myheader
            \endfirsthead

            \longtablecaptions{3}{}

            \hline

            \multirowcell{3}{3cm}{Not Downregulated (or overall low expression)}
             & CCNE2  & ENSG00000175305 & Overall    & low  & 5.34E-04 & 8.64E-03 \\
            \hhline{~------}
             & MMP2   & ENSG00000087245 & Prog. Free & high & 2.29E-05 & 2.32E-03 \\
            \hhline{~------}
             & OSMR   & ENSG00000145623 & Prog. Free & high & 5.67E-04 & 7.15E-03 \\
            \hhline{~------}
            \hline

            \multirowcell{8}{2.7cm}{Continuously Downregulated (BMPC > MGUS > sMM > MM > MMR)}
             & AXL    & ENSG00000167601 & Overall    & high & 3.64E-05 & 1.84E-03 \\
            \hhline{~------}
             & COL1A1 & ENSG00000108821 & Prog. Free & high & 3.03E-04 & 4.37E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 5.93E-04 & 8.64E-03 \\
            \hhline{~------}
             & CXCL12 & ENSG00000107562 & Prog. Free & high & 1.16E-04 & 2.93E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 6.48E-04 & 8.64E-03 \\
            \hhline{~------}
             & CYP1B1 & ENSG00000138061 & Overall    & high & 6.84E-04 & 8.64E-03 \\
            \hhline{~------}
             & DCN    & ENSG00000011465 & Overall    & high & 2.47E-04 & 8.33E-03 \\
            \hhline{~------}
             & LRP1   & ENSG00000123384 & Overall    & high & 4.34E-04 & 8.64E-03 \\
            \hhline{~------}
             & LTBP2  & ENSG00000119681 & Prog. Free & high & 9.03E-05 & 2.93E-03 \\
            \hhline{~------}
             & CYP1B1 & ENSG00000138061 & Overall    & high & 6.84E-04 & 8.64E-03 \\
            \hhline{~------}
             & DCN    & ENSG00000011465 & Overall    & high & 2.47E-04 & 8.33E-03 \\
            \hhline{~------}
             & LRP1   & ENSG00000123384 & Overall    & high & 4.34E-04 & 8.64E-03 \\
            \hhline{~------}
             & LTBP2  & ENSG00000119681 & Prog. Free & high & 9.03E-05 & 2.93E-03 \\
            \hhline{~------}
             & MFAP5  & ENSG00000197614 & Prog. Free & high & 2.43E-04 & 4.09E-03 \\
            \hhline{~------}
             & MMP14  & ENSG00000157227 & Prog. Free & high & 6.93E-05 & 2.93E-03 \\
            \hhline{~------}
             & MYL9   & ENSG00000101335 & Prog. Free & high & 1.46E-04 & 2.95E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 1.56E-05 & 1.57E-03 \\
            \hline
        \end{longtable}
    }


Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% ## Fig. 7
\includeimage[1][{\fmleft} 6.8in {\fmright} {\fmtop}]{
    FIGS/figures9_7.pdf
}\captionof{figure}{
    Proposed model of “Detached Daughter Driven Dissemination” (DDDD) in
    aggregating multiple myeloma. \tile{Heterotypic Interaction} Malignant
    plasma cells colonize the bone marrow microenvironment by adhering to an MSC
    (or osteoblast, ECM, etc.) to maximize growth and survival through paracrine
    and adhesion mediated signaling, even if contact may trigger initial
    apoptosis. Gene expression will focus on establishing a strong anchor within
    the bone marrow, but also on attracting other myeloma cells (via secretion
    of ECM factors and CXCL12/CXCL8, respectively). \tile{Cell Division} Cell
    fission can generate one daughter cell that no longer adheres to the MSC
    (nMA). \tile{Homotypic Interaction} If myeloma cells have the capacity to
    grow as aggregates, the daughter cell stays attached to their MSC-adhering
    mother cell (MA). \tile{Re-Adhesion} The daughter cell “rolls around” the
    mother cell until it re-adheres to the MSC. Our model estimates the rolling
    duration to be \SIrange{1}{10}{\hour} long. \tile{Proliferation \& Saturation} We
    estimate that a single myeloma cell covers one MSC completely after roughly
    four population doublings. When heterotypic adhesion is saturated,
    subsequent daughter cells benefit from a homotypic interaction, since they
    stay close to growth-factor secreting MSCs and focus gene expression on
    proliferation (e.g. driven by E2F) and not adhesion (driven by NF-κB).
    \tile{Critical Size} Homotypic interaction is weaker than heterotypic
    interaction, and each cell fission destabilizes the aggregate. Hence,
    detachment of myeloma cells may depend mostly on aggregate size.
    \tile{Dissemination} After myeloma cells have detached, they gained a
    viability advantage through IL-6-independence (with unknown duration), which
    enhances their survival outside of the bone marrow and allows them to spread
    throughout the body. }\label{fig:7}



Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% lorem ipsum

% V-Well is described in \refapdx{apdx:supplemental}{fig:S1}

% V-Well is described in \refapdx{apdx:supplemental}{fig:S1}, \ref{fig:S2}

% % Image cytometry is described in \autoref{subapdx:sup_figtabs} \autoref{fig:S2}

% Full analysis is shown in \refapdx{subapdx:example_analysis}

% % ======================================================================
% % == Discussion
% % ======================================================================
\unnsubsection{Discussion}\label{C1:discussion} % > Require unique label
\ %
lorem ipsum

% % == Paper 1 ===========================================================
% % > You could import .pdf here, but chapter based theses should apply the 
% % > manuscripts into the formatting of the thesis
% % \addpdf{Research Article: Cancer Research Communications}{PUBLICATIONS/AACR.pdf}





