



% ======================================================================
% == Chapter 1
% ======================================================================
\unnsection{Chapter 1: Modelling Myeloma Dissemination \textit{in vitro}}
\vspace{-\baselineskip} % > Remove space made by empty lines

% ## Reset reference counters of figs, tabs, so each chapter starts at 1
\setcounter{figure}{0}
\setcounter{table}{0}

% Title: Keep it Together: Modelling Myeloma Dissemination in vitro with
% hMSC- Interacting Subpopulations of INA-6 Cells and their
% Aggregation/Detachment Dynamics 

% ======================================================================
% == Abstract
% ======================================================================



% > First argument is the label
% > Second argument is the header name
% > Third argument is the abstract text
\customabstract{c1:abstract}{Abstract}{ %
    Multiple myeloma involves early dissemination of malignant plasma cells
    across the bone marrow; however, the initial steps of dissemination remain
    unclear. Human bone marrow- derived mesenchymal stromal cells (hMSCs)
    stimulate myeloma cell expansion (e.g., IL-6) and simultaneously retain
    myeloma cells via chemokines (e.g., CXCL12) and adhesion factors. Hence, we
    hypothesized that the imbalance between cell division and retention drives
    dissemination. We present an \textit{in vitro} model using primary hMSCs co-cultured
    with INA-6 myeloma cells. Time-lapse microscopy revealed proliferation and
    attachment/detachment dynamics. Separation techniques (V-well adhesion assay
    and well plate sandwich centrifugation) were established to isolate
    MSC-interacting myeloma subpopulations that were characterized by RNAseq,
    cell viability and apoptosis. Results were correlated with gene expression
    data ($n=837$) and survival of myeloma patients ($n=536$). On dispersed
    hMSCs, INA-6 saturate hMSC-surface before proliferating into large homotypic
    aggregates, from which single cells detached completely. On confluent hMSCs,
    aggregates were replaced by strong heterotypic hMSC-INA-6 interactions,
    which modulated apoptosis time-dependently. Only INA-6 daughter cells
    (\nMAina) detached from hMSCs by cell division but sustained adherence to
    hMSC-adhering mother cells (\MAina). Isolated \nMAina indicated
    hMSC-autonomy through superior viability after IL-6 withdrawal and
    upregulation of proliferation-related genes. \MAina upregulated adhesion and
    retention factors (CXCL12), that, intriguingly, were highly expressed in
    myeloma samples from patients with longer overall and progression-free
    survival, but their expression decreased in relapsed myeloma samples.
    Altogether, \textit{in vitro} dissemination of INA-6 is driven by detaching daughter
    cells after a cycle of hMSC-(re)attachment and proliferation, involving
    adhesion factors that represent a bone marrow-retentive phenotype with
    potential clinical relevance. %
}

% ## Statement of Significance
\subsection*{Statement of Significance}
Novel methods describe \textit{in vitro} dissemination of myeloma cells as detachment of
daughter cells after cell division. Myeloma adhesion genes were identified that
counteract \textit{in vitro} detachment with potential clinical relevance.
\newpage


% ======================================================================
% == Introduction
% ======================================================================
% \unnsubsection{C1:Introduction}{Introduction}
\unnsubsection{Introduction}\label{C1:introduction} % > Require unique label
\ %
Multiple myeloma arises from clonal expansion of malignant plasma cells in the
bone marrow (BM). At diagnosis, myeloma cells have disseminated to multiple
sites in the skeleton and, in some cases, to “virtually any tissue”
\cite{bladeExtramedullaryDiseaseMultiple2022,rajkumarInternationalMyelomaWorking2014}.
However, the mechanism through which myeloma cells initially disseminate remains
unclear. Dissemination is a multistep process involving invasion, intravasation,
intravascular arrest, extravasation, and colonization
\cite{zeissigTumourDisseminationMultiple2020}. To initiate dissemination,
myeloma cells overcome adhesion, retention, and dependency on the BM
microenvironment, which could involve the loss of adhesion factors such as CD138
\cite{akhmetzyanovaDynamicCD138Surface2020,garcia-ortizRoleTumorMicroenvironment2021}.
BM retention is mediated by multiple factors: First, chemokines (CXCL12 and
CXCL8) produced by mesenchymal stromal cells (MSCs), which attract plasma cells
and prime their cytoskeleton and integrins for adhesion
\cite{aggarwalChemokinesMultipleMyeloma2006,alsayedMechanismsRegulationCXCR42007}.
Second, myeloma cells must overcome the anchorage and physical boundaries of the
extracellular matrix (ECM), consisting of e.g. fibronectin, collagens, and
proteoglycans such as decorin
\cite{huDecorinmediatedSuppressionTumorigenesis2021,huangHigherDecorinLevels2015,katzAdhesionMoleculesLifelines2010,kiblerAdhesiveInteractionsHuman1998}.
Simultaneously, ECM provides signals inducing myeloma cell cycle arrest or
progression the cell cycle
\cite{huDecorinmediatedSuppressionTumorigenesis2021,katzAdhesionMoleculesLifelines2010}.
\ %
ECM is also prone to degradation, which is common in several osteotropic
cancers, and is the cause of osteolytic bone disease. This is driven by a
‘vicious cycle’ that maximizes bone destruction by extracting growth factors
(EGF and TGF-β) that are stored in calcified tissues
\cite{glaveyProteomicCharacterizationHuman2017}. Third, direct contact with MSCs
physically anchors myeloma cells to the BM
\cite{zeissigTumourDisseminationMultiple2020,sanz-rodriguezCharacterizationVLA4dependentMyeloma1999}.
Fourth, to disseminate to distant sites, myeloma cells require, at least
partially, independence from essential growth and survival signals provided by
MSCs in the form of soluble factors or cell adhesion signaling
\cite{garcia-ortizRoleTumorMicroenvironment2021, chatterjeePresenceBoneMarrow2002,hideshimaUnderstandingMultipleMyeloma2007}.
For example, the VLA4 (Myeloma)–VCAM1 (MSC)-interface activates NF-κB in both
myeloma and MSCs, inducing IL-6 expression in MSCs. The independence from MSCs
is then acquired through autocrine survival signaling
\cite{frassanitoAutocrineInterleukin6Production2001, urashimaCD40LigandTriggered1995}.
In short, anchorage of myeloma cells to MSCs or ECM is a ‘double-edged sword’:
adhesion counteracts dissemination, but also presents signaling cues for growth,
survival, and drug resistance \cite{solimandoDrugResistanceMultiple2022}.


To address this ambiguity, we developed an \textit{in vitro} co-culture system
modeling diverse adhesion modalities to study dissemination, growth, and
survival of myeloma cells and hMSCs. Co-cultures of hMSCs and the myeloma cell
line INA-6 replicated tight interactions and aggregate growth, akin to
``microtumors'' in Ghobrial’s metastasis concept
\cite{ghobrialMyelomaModelProcess2012}. We characterized the growth
conformations of hMSCs and INA-6 as homotypic aggregation \textit{vs.}
heterotypic hMSC adherence and their effects on myeloma cell survival. We
tracked INA-6 detachments from aggregates and hMSCs, thereby identifying a
potential ``disseminated'' subpopulation lacking strong adhesion. We developed
innovative techniques (V-well adhesion assay and well plate sandwich
centrifugation) to separate weakly and strongly adherent subpopulations for the
subsequent analysis of differential gene expression and cell survival. Notably,
our strategy resolves the differences in gene expression and growth behavior
between cells of one cell population in ``direct'' contact with MSCs. In
contrast, previous methods differentiated between ``direct'' and ``indirect''
cell-cell contact using transwell inserts
\cite{dziadowiczBoneMarrowStromaInduced2022}. To evaluate whether genes
mediating adhesion and growth characteristics of INA-6 were associated with
patient survival, we analyzed publicly available datasets
\cite{seckingerTargetExpressionGeneration2017b,seckingerCD38ImmunotherapeuticTarget2018}.


\newpage


% ======================================================================
% == Methods
% ======================================================================
\unnsubsection{Materials and Methods}\label{C1:methods}
\ %
See \refapdx{subapdx:methods} for a complete method list and description.


\subsubsection*{Ethics Statement}
\ %
Primary human MSCs were collected with the written informed consent of all
patients. The procedure was conducted in accordance with recognized ethical
guidelines (Helsinki Declaration) and approved by the local Ethics Committee of
the University of Würzburg (186/18).



\subsubsection*{Cultivation and Co-Culturing of primary hMSCs and INA-6}
\ %
Primary human MSCs were obtained from the femoral head of 34 non-myeloma
patients (\refapdx{apdx:supplemental}{tab:S1}: 21 male and 13 female, mean age
68.9 ± 10.6) undergoing elective hip arthroplasty. The INA-6 cell line
(\textit{DSMZ Cat\# ACC-862}, \textit{RRID:CVCL\_5209},
\href{https://www.cellosaurus.org/CVCL_5209}{link}) was initially isolated from
a pleural effusion sample obtained from an 80-year-old male with multiple
myeloma \cite{burgerGp130RasMediated2001c,gramatzkiTwoNewInterleukin61994}.
hMSCs were not tested for mycoplasma, whereas stocks of INA-6 were tested in
this study (\refapdx{apdx:supplemental}{tab:S1}) using the \textit{Venor GeM OneStep} kit
(Minerva Biolabs, Berlin, Germany). For each co-culture, hMSCs were seeded
\SI{24}{\hour} before INA-6 addition to generate the MSC-conditioned medium
(CM). INA-6 cells were washed with PBS, resuspended in MSC medium, and added to
hMSCs so that the co-culture comprised \SI{33}{\percent} (v/v) of CM gathered
directly from the respective hMSC donor. The co-cultures were not substituted
for IL-6 \cite{chatterjeePresenceBoneMarrow2002}.

% 132 Cell Viability and Apoptosis Assay
% 133 Cell viability and apoptosis rates were measured using CellTiter-Glo Luminescent Cell
% 134 Viability Assay and Caspase-Glo 3/7 Assay, respectively (Promega GmbH, Mannheim,
% 135 Germany).



\subsubsection*{Cell Viability and Apoptosis Assay}
\ %
Cell viability and apoptosis rates were measured using \textit{CellTiter-Glo Luminescent
    Cell Viability Assay} and \textit{Caspase-Glo 3/7 Assay}, respectively (Promega GmbH,
Mannheim, Germany).


% 136 Automated Fluorescence Microscopy
% 137 Microscopic images were acquired using an Axio Observer 7 (Zeiss) with a COLIBRI LED
% 138 light source and motorized stage top using 5x and 10x magnification. The tiled images had
% 139 an automatic 8-10% overlap and were not stitched.

\subsubsection*{Automated Fluorescence Microscopy}
\ %
Microscopic images were acquired using an Axio Observer 7 (Zeiss) with a COLIBRI LED
light source and motorized stage top using 5x and 10x magnification. The tiled images had
an automatic \SIrange{8}{10}{\percent} overlap and were not stitched.


% 140 Live Cell Imaging
% 141 hMSCs (stained with PKH26) were placed into an ibidi Stage Top Incubation System and
% 142 equilibrated to 80% humidity and 5% CO2. INA-6 (2 × 103 cells/cm2) were added directly
% 143 before the start of acquisition. Brightfield and fluorescence images of up to 13 mm2 of the co-
% 144 culture area were acquired every 15 min for 63 h. Each event of interest was manually
% 145 analyzed and categorized into defined event parameters.

\subsubsection*{Live Cell Imaging}
hMSCs (stained with PKH26) were placed into an ibidi Stage Top Incubation System
and equilibrated to \SI{80}{\percent} humidity and \SI{5}{\percent}
CO\textsubscript{2}. INA-6 (2 $\times$ 10\textsuperscript{3}
cells/cm\textsuperscript{2}) were added directly before the start of
acquisition. Brightfield and fluorescence images of up to \SI{13}{\mm\squared}
of the co-culture area were acquired every \SI{15}{\minute} for \SI{63}{\hour}.
Each event of interest was manually analyzed and categorized into defined event
parameters.


% 146 V-well Adhesion Assay
% 147 INA-6 cells were arrested during mitosis by two treatments with thymidine, followed by
% 148 nocodazole. Arrested INA-6 were released and added to 96 V-well plates (104 cells/cm2) on
% 149 top of confluent hMSCs and adhered for 1-3 h. The co-culture was stained with calcein-AM
% 150 (Thermo Fisher Scientific, Darmstadt, Germany) before non-adherent INA-6 were pelleted
% 151 into the tip of the V-well (2000 rpm, 5-10 min). MSC-adhering INA-6 cells were manually
% 152 detached by rapid pipetting. The pellet brightness was measured microscopically and the
% 153 pellet was isolated by pipetting.


\subsubsection*{V-well Adhesion Assay}
\ %
INA-6 cells were arrested during mitosis by two treatments with thymidine, followed by
nocodazole. Arrested INA-6 were released and added to 96 V-well plates (10\textsuperscript{4}
cells/cm\textsuperscript{2}) on
top of confluent hMSCs and adhered for \SIrange{1}{3}{\hour}. The co-culture was stained with calcein-AM
(Thermo Fisher Scientific, Darmstadt, Germany) before non-adherent INA-6 were pelleted
into the tip of the V-well (\SI{2000}{rpm}, \SIrange{5}{10}{\minute}). MSC-adhering INA-6 cells were manually
detached by rapid pipetting. The pellet brightness was measured microscopically and the
pellet was isolated by pipetting.

% 154 Cell Cycle Profiling by Image Cytometry
% 155 Isolated INA-6 cells were fixed in 70% ice-cold ethanol, washed, resuspended in PBS,
% 156 distributed in 96-well plates, and stained with Hoechst 33342. The plates were scanned at 5x
% 157 magnification. A pre-trained convolutional neural network (Intellesis, Zeiss) was fine-tuned to
% 158 segment the scans into single nuclei and exclude fragmented nuclei. Nuclei were filtered to
% 159 exclude extremes of size roundness. The G0/G1 frequency was determined by Gaussian
% 160 curve fitting.

\subsubsection*{Cell Cycle Profiling by Image Cytometry}
\ %
Isolated INA-6 cells were fixed in \SI{70}{\percent} ice-cold ethanol, washed, resuspended in PBS,
distributed in 96-well plates, and stained with Hoechst 33342. The plates were scanned at 5x
magnification. A pre-trained convolutional neural network (Intellesis, Zeiss) was fine-tuned to
segment the scans into single nuclei and exclude fragmented nuclei. Nuclei were filtered to
exclude extremes of size and roundness. The G0/G1 frequency was determined by Gaussian
curve fitting.


\subsubsection*{Well Plate Sandwich Centrifugation (WPSC)}
\ %
hMSCs were grown to confluence in 96-well plates coated with collagen I (rat
tail; Corning, NY, USA). INA-6 cells were added and the cells were allowed to
adhere for \SI{24}{\hour}. A second plate (``catching plate'') was attached
upside down to the top of the co-culture plate. That ``well plate sandwich'' was
turned around and the content of the co-culture plate was centrifuged into the
catching plate three times (\SI{40}{\second} at \SI{110}{g}) while gently adding
\SI{30}{\micro\liter} of medium in between centrifugation steps.
Non-MSC-adhering INA-6 cells were collected from the catching plate, whereas
MSC-adhering INA-6 cells were isolated by digesting the co- culture with
accutase. For RNA sequencing (RNAseq), all samples were purified using anti-CD45
magnetic-assisted cell sorting (Miltenyi Biotec B.V. \& Co. KG, Bergisch
Gladbach).

\subsubsection*{RNA Isolation}
\ %
RNA was isolated using the \textit{NucleoSpin RNA II Purification Kit}
(Macherey-Nagel) according to the manufacturer's instructions. RNA was isolated
from INA-6 cells co-cultured with a unique hMSC donor ($n=5$ for qPCR, $n=11$
for RNA sequencing).

\subsubsection*{RNA sequencing, Differential Expression, and Functional Enrichment Analysis}
\ %
RNA sequencing (RNAseq) was performed at the Core Unit Systems Medicine,
University of Würzburg. mRNA was enriched with polyA beads. Fastq files were
aligned to the GRCh38 reference genome using \texttt{STAR}
(\textit{RRID:SCR\_004463},
\href{https://scicrunch.org/resolver/RRID:SCR_004463}{link}) and raw read
counts were generated using \texttt{HTseq} (\textit{RRID:SCR\_005514},
\href{https://scicrunch.org/resolver/SCR_005514}{link})
\cite{andersHTSeqPythonFramework2015,dobinSTARUltrafastUniversal2013,zerbinoEnsembl20182018}.
Differential gene expression was analyzed using \texttt{edgeR} in R (version
3.6.3) (\textit{RRID:SCR\_012802},
\href{https://scicrunch.org/resolver/SCR_012802}{link}). Functional
enrichment analysis was performed using \texttt{Metascape}
(\textit{RRID:SCR\_016620},
\href{https://scicrunch.org/resolver/SCR_016620}{link})
\cite{zhouMetascapeProvidesBiologistoriented2019}.


% 182 RT-qPCR
% 183 RNA (1 μg) was reverse transcribed using SuperScript IV reverse transcriptase (Thermo
% 184 Fisher Scientific). qPCR was performed using 10 μL GoTaq qPCR Master Mix (Promega),
% 185 1:10 diluted cDNA, and 5 pmol of primers obtained from Biomers.net or Qiagen
% 186 (Supplementary Table 3).
\subsubsection*{RT-qPCR}
\ %
RNA (\SI{1}{\micro\gram}) was reverse transcribed using \textit{SuperScript IV
    reverse transcriptase} (Thermo Fisher Scientific). qPCR was performed using
\SI{10}{\micro\liter} \textit{GoTaq qPCR Master Mix} (Promega), \SI{1}{:10}
diluted cDNA, and \SI{5}{pmol} of primers obtained from Biomers.net or Qiagen
(\refapdx{apdx:supplemental}{tab:S3}).


% 187 Statistics
% 188 Inferential statistics were performed using Python (IPython, RRID:SCR_001658, link) (3.10)
% 189 packages pingouin (0.5.1) and statsmodels (0.14.0) (29, 30). The figures were plotted using
% 190 plotastic (0.0.1) (31). Normality (for n ≥ 4) and sphericity were ensured using Mauchly’s and
% 191 Shapiro-Wilk tests, respectively. Data points were log10 transformed to convert the scale
% 192 from multiplicative to additive or to fulfill sphericity requirements. p-value = 0.05 > * > 0.01 >
% 193 ** > 10-3 > *** 10-4 > ****. P-values were either adjusted (p-adj) or not adjusted (p-unc) for
% 194 family wise error rate. Power calculations were not performed to determine the sample size.
\subsubsection*{Statistics}
\ %
Inferential statistics were performed using Python (IPython,
\textit{RRID:SCR\_001658},
\href{https://www.ncbi.nlm.nih.gov/sra?term=SCR_001658}{link}) (3.10) packages
\textit{pingouin} (0.5.1) and \texttt{statsmodels} (0.14.0)
\cite{vallatPingouinStatisticsPython2018,seaboldStatsmodelsEconometricStatistical2010}.
The figures were plotted using \texttt{plotastic} (0.0.1)
\cite{kuricPlotasticBridgingPlotting2024}. Normality (for \( n \geq 4 \)) and
sphericity were ensured using Mauchly’s and Shapiro-Wilk tests, respectively.
Data points were log\(_{10}\) transformed to convert the scale from
multiplicative to additive or to fulfill sphericity requirements. \( p = 0.05 >
\text{*} > 0.01 > \text{**} > 10^{-3} > \text{***}> 10^{-4} > \text{****} \).
$p$-values were either adjusted (\( p{\text{-adj}} \)) or not adjusted (\(
p{\text{-unc}} \)) for family wise error rate. Power calculations were not
performed to determine the sample size.


% 195 Patient Cohort, Analysis of Survival and Expression
% 196 Survival and gene expression data were obtained as previously described (21, 22) and are
% 197 available at the European Nucleotide Archive (ENA) under accession numbers PRJEB36223
% 198 and PRJEB37100. The expression level was categorized into “high” and “low” using maxstat
% 199 (Maximally selected Rank Statistics) thresholds (32).
% 200 Data Availability Statement
% 201 A detailed description of the methods is provided in the Supplementary Material section. Raw
% 202 tabular data and examples of analyses and videos are available in the github repository, link.
% 203 Raw RNAseq data are available from the NCBI Gene Expression Omnibus (GEO)
% 204 (RRID:SCR_005012, link) (GSE261423). Microscopy data are available at BioStudies
% 205 (EMBL-EBI) (RRID:SCR_004727, link) (S-BIAD1092).

\subsubsection*{Patient Cohort, Analysis of Survival and Expression}
\ %
Survival and gene expression data were obtained as previously described
\cite{seckingerTargetExpressionGeneration2017b,seckingerCD38ImmunotherapeuticTarget2018}
and are available at the European Nucleotide Archive (ENA) under accession
numbers PRJEB36223 and PRJEB37100. The expression level was categorized into
``high'' and ``low'' using \texttt{maxstat} (Maximally selected Rank Statistics)
thresholds \cite{hothornMaximallySelectedRank}.

% 200 Data Availability Statement
% 201 A detailed description of the methods is provided in the Supplementary Material section. Raw
% 202 tabular data and examples of analyses and videos are available in the github repository, link.
% 203 Raw RNAseq data are available from the NCBI Gene Expression Omnibus (GEO)
% 204 (RRID:SCR_005012, link) (GSE261423). Microscopy data are available at BioStudies
% 205 (EMBL-EBI) (RRID:SCR_004727, link) (S-BIAD1092).

\subsubsection*{Data Availability Statement}
\ %
A detailed description of the methods is provided in the Supplementary Material section. Raw
tabular data and examples of analyses and videos are available in the github repository, \href{https://github.com/markur4/Supplemental-INA-6-Subpopulations-and-Aggregation-Detachment-Dynamics}{link}.
Raw RNAseq data are available from the NCBI Gene Expression Omnibus (GEO)
(\textit{RRID:SCR\_005012}, \href{https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE261423}{link}) (GSE261423). Microscopy data are available at BioStudies
(EMBL-EBI) (\textit{RRID:SCR\_004727}, \href{https://www.ebi.ac.uk/biostudies/bioimages/studies/S-BIAD1092?key=69bafe9c-74ff-492b-9e68-bd42655c4d1b}{link}) (S-BIAD1092).

\newpage






% ======================================================================
% == Results
% ======================================================================
\unnsubsection{Results}\label{C1:results}
\ %
\subsubsection*{\INA Cells Saturate hMSC-Interaction to Proliferate into Aggregates}
hMSCs are isolated as a heterogeneous cell population. To analyze whether \INA
cells could adhere to every hMSC, we saturated hMSCs with \INA. A seeding ratio
of 1:4 (hMSC:\INA) resulted in the occupation of \SIplusminus{93}{6}{\percent}
of single hMSCs by one or more \INA cells within 24 hours after \INA addition,
escalating to \SI{6}{\percent} after \SI{48}{hours} (\autoref{fig:1}A, B).
Therefore, most hMSCs provide an interacting surface for \INA cells.

INA-6 exhibits homotypic aggregation when cultured alone, a phenomenon observed
in some freshly isolated myeloma samples (up to \SI{100}{cells} after
\SI{6}{hours})
\cite{kawanoHomotypicCellAggregations1991,okunoVitroGrowthPattern1991}. Adding
hMSCs at a 1:1 ratio led to smaller aggregates after \SI{24}{hours} (size
\SIrange{1}{5}{cells}), all of which were distributed over
\SIplusminus{52}{2}{\percent} of all hMSCs (\autoref{fig:1}A, B). Intriguingly,
\INA aggregation was notably absent when grown on confluent hMSCs, and occurred
only when heterotypic interactions were limited to 0.2 hMSCs per \INA cell
(\autoref{fig:1}C). We concluded that \INA cells prioritize heterotypic over
homotypic interactions.

To monitor the formation of such aggregates, we conducted live-cell imaging of
hMSC/\INA co-cultures for \SI{63}{hours}. We observed that \INA cells adhered
long after cytokinesis, constituting \SIplusminus{55}{12}{\percent} of all
homotypic interactions between \SI{13}{hours} and \SI{26}{hours}, increasing to
more than \SI{75}{\percent} for the remainder of the co-culture
(\autoref{fig:1}D). Therefore, homotypic \INA aggregates were mostly formed by
cell division.

\subsubsection*{Apoptosis of \INA Depends on Ratio Between Heterotypic and Homotypic Interaction}
Although direct interaction with hMSCs has been shown to enhance myeloma cell
survival through NF-$\kappa$B signaling
\cite{hideshimaUnderstandingMultipleMyeloma2007}, the impact of aggregation on
myeloma cell viability during hMSC interaction remains unclear. To address this,
we measured the cell viability (ATP) and apoptosis rates of \INA cells growing
as homotypic aggregates compared to those in heterotypic interactions with hMSCs
by modulating hMSC density (\autoref{fig:1}E). To equalize the background
signaling caused by soluble MSC-derived factors, all cultures were incubated in
hMSC-conditioned medium and the results were normalized to \INA cells cultured
without direct hMSC contact (\autoref{fig:1}E, left).

\newpage

% ## Example:
% > \includeimage[scale][left bottom right top]{
% >     pathtopicture
% > }{caption}

% ## Fig. 1
% > use \noindent, for some reason first image is indented otherwise
\noindent\includeimage[1][{\fmleft} 2.8in {\fmright} {\fmtop}]{
    FIGS/figures9_1.pdf
}\splitcaption{ %
    INA-6 growth conformations and survival on hMSCs. \tile{A} Interaction of
    INA-6 (green) with hMSCs (black, negative staining) at different INA-6
    densities (constant hMSC densities). \mbox{Scale bar = \SI{200}{\um}}.
    \tile{B} Frequency of single hMSCs (same as A) that are covered by INA-6 of
    varying group sizes. Technical replicates = three per datapoint; Single
    hMSCs evaluated: 100 per technical replicate. \tile{C} Interaction of INA-6
    with hMSCs at different hMSC densities (constant INA-6 densities). Scale bar
    = \SI{300}{\um}. 
}{%
    \tile{D} Two types of homotypic interaction: Attachment after cell contact
    and sustained attachment of daughter cells after cell division. Datapoints
    represent one of four independent time-lapse recordings, each evaluating 116
    interaction events. \tile{E} Effects of hMSC-density on the viability (ATP,
    top) and apoptosis (Caspase3/7 activity, bottom). INA-6:MSC ratio = 4:1;
    \mbox{Technical replicates = four per datapoint}; E left: Signals were
    measured in INA-6 washed off from hMSCs and normalized by INA-6 cultured in
    MSC-conditioned medium ($= \text{red line}$) ($n=4$). E right: Signals were
    measured in co-cultures and normalized by the sum of the signals measured in
    hMSC and INA-6 cultured separately ($= \text{red line}$) ($n=3$).
    \tile{Statistics} Paired t-test, two-factor RM- ANOVA. Datapoints represent
    independent co-cultures with hMSCs from three (A, B, D, E right), four (E
    left) unique donors. Confl. = Confluent. }
\label{fig:1}


\INA viability (ATP) was not affected by the direct adhesion of hMSCs at any
density. However, apoptosis rates decreased over time [\(F(2, 6) = 23.29\),
\(p\)-unc = \(1.49 \times 10^{-3}\), Two-factor RM-ANOVA], interacting
significantly with MSC density [\(F(4, 12) = 6.98\), \(p\)-unc = \(3.83 \times
10^{-3}\)]. For example, 24 hours of adhesion to confluent MSCs increased
apoptosis rates by \SIplusminus{1.46}{0.37}{fold}, while culturing \INA cells on
dispersed hMSCs (ratio 1:1) did not change the apoptosis rate
(\SIplusminus{1.01}{0.26}{fold}).


We presumed that sensitive apoptotic cells might have been lost when harvesting
\INA cells from hMSCs. Hence, we measured survival parameters in the co-culture
and in hMSC and \INA cells cultured separately (\autoref{fig:1}E, right). We
defined MSC interaction effects when the survival measured in the co-culture
differed from the sum of the signals measured from \INA and hMSCs alone.
RM-ANOVA confirmed that adherence to confluent MSCs increased apoptosis rates of
\INA cells \SI{24}{hours} after adhesion and decreased after \SI{72}{hours}
[interaction between MSC density and time: \(F(2, 4) = 26.86\), \(p\)-unc =
\(4.80 \times 10^{-3}\), Two-factor RM-ANOVA], whereas \INA cells were
unaffected when grown on dispersed hMSCs.

In summary, the growth conformation of \INA cells, measured as the ratio between
homotypic aggregation and heterotypic MSC interactions, affected apoptosis rates
of \INA cells.



\subsubsection*{Single \INA Cells Detach Spontaneously from Aggregates of Critical Size}
Using time-lapse microscopy, we observed that \SIplusminus{26}{8}{\percent} of
\INA aggregates growing on single hMSCs spontaneously shed \INA cells
(\autoref{fig:2}A, B; Supplementary Video 1). Notably, all detached cells
exhibited similar directional movements, suggesting entrainment in convective
streams generated by temperature gradients within the incubation chamber. \INA
predominantly detached from other \INA cells or aggregates (\autoref{fig:2}C),
indicating weaker adhesive forces in homotypic interactions than in heterotypic
interactions. The detachment frequency increased after \SI{52}{hours}, when most
aggregates that shed \INA cells were categorized as large (greater than 30
cells) (\autoref{fig:2}D). Since approximately 10-20 \INA cells already fully
covered a single hMSC, we suggest that myeloma cell detachment depended not only
on hMSC saturation but also required a minimum aggregate size. Interestingly,
\INA detached mostly as single cells, independent of aggregate size categories
[\(F(2, 6) = 4.68\), \(p\)-unc = 0.059, Two-factor RM-ANOVA] (\autoref{fig:2}E),
showing that aggregates remained mostly stable despite losing cells.


% ## Fig. 2
\includeimage[1][{\fmleft} 7.4in {\fmright} {\fmtop}]{
    FIGS/figures9_2.pdf
}\captionof{figure}{ %
    Time-lapse analysis of INA-6 detachment from INA-6 aggregates and hMSCs.
    \tile{A} Frequency of observed INA-6 aggregates that did or did not lose
    INA-6 cell(s). 87 aggregates were evaluated per datapoint. \tile{B} Example
    of a ``disseminating`` INA-6 aggregate growing on fluorescently (PKH26)
    stained hMSC (from A-D). Dashed green lines are trajectories of detached
    INA-6 cells. Scale bar = \SI{50}{\um}. \tile{\mbox{C-E}} Quantitative assessment of
    INA-6 detachments. 45 detachment events were evaluated per datapoint.
    Seeding ratio INA-6:MSC = 4:1. \tile{C}
    Most INA-6 cells dissociated from another INA-6 cell and not from an hMSC
        [$F(1, 3) = 298$, $p\text{-unc}=4.2e-4$]. \tile{D} Detachment frequency of
    aggregate size categories. \tile{E} Detachment frequency of INA-6 cells
    detaching as single, pairs or more than three cells. \tile{Statistics} (A):
    Paired-t-test; (C-E): Paired-t-test, Two-factor RM-ANOVA; Datapoints
    represent three (A) or four (C-E) independent time-lapse recordings of
    co-cultures with hMSCs from two (A) or three (C-E) unique donors.
}\label{fig:2}






% ## Fig. 3
\includeimage[1][{\fmleft} 5.4in {\fmright} {\fmtop}]{
    FIGS/figures9_3.pdf
}\captionof{figure}{ %
    Detachment of INA-6 daughter cells after Cell Division. \tile{A-D} INA-6
    divisions in interaction with confluent hMSCs. Seeding ratio INA-6:MSC = 4:20.
    \tile{A} Three examples of dividing INA-6 cells generating either two MA, or one
    MA and one nMA daughter cells as described in (G). Dashed circles mark mother
    cells (white), MA cell (blue), and first position of nMA cell (green). Scale
    bar: \SI{20}{\um}. \tile{B} Cell division of MSC-adhering (MA) mother cell can
    yield one mobile non-MSC-adhering (nMA) daughter cell. \tile{C} Frequencies of
    INA-6 pairs defined in (A, B) per observed cell division. 65 divisions were
    evaluated for each of three independent time-lapse recordings. \tile{D} Rolling
    duration of nMA cells after division did not depend on hMSC donor [$H(2) =
                5.250$, $p\text{-unc} = .072$]. Datapoints represent single nMA-cells after
    division. \tile{E-G} Adhesive and cell cycle assessment of MSC-interacting INA-6
    subpopulations using the V-Well assay. \tile{E} Schematic of V-Well Assay (see
    \refapdx{apdx:supplemental}{fig:S1} for detailed analysis). MSC-interacting
    subpopulations were separated by subsequent centrifugation and removal of the
    pellet. The pellet size was quantified by its total fluorescence brightness.
    Adhering subpopulations were resuspended by rough pipetting. \tile{F} Relative
    cell pellet sizes of adhesive INA-6 subpopulations that cycle either
    asynchronously or were synchronized at mitosis. Gray lines in-between points
    connect dependent measurements of co-cultures ($n=9$) that shared the same
    hMSC-donor and INA-6 culture. Co-cultures were incubated for three different
    durations (\SIlist{1;2;3}{\hour} after INA-6 addition). Time points were pooled,
    since time did not show an effect on cell adhesion [$F(2,4) = 1.414$,
            $p\text{-unc} = 0.343$]. Factorial RM-ANOVA shows an interaction between cell
    cycle and the kind of adhesive subpopulation [$F(1, 8) = 42.67$, $p\text{-unc} =
                1.82e-04$]. Technical replicates = 4 per datapoint. \tile{G} Cell cycles were
    profiled in cells gathered from the pellets of four independent co-cultures
    ($n=4$) and the frequency of G0/G1 cells are displayed depending on co-culture
    duration (see \refapdx{apdx:supplemental}{fig:S3} for cell cycle profiles). Four
    technical replicates were pooled after pelleting. \tile{Statistics} (D):
    Kruskal-Wallis H-test. (F): Paired t-test, (G): Paired t-test, two-factor
    RM-ANOVA. Datapoints represent INA-6 from independent co- cultures with hMSCs
    from three unique donors. }\label{fig:3}


Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% ## Fig. 4
\includeimage[1][{\fmleft} 4.3in {\fmright} {\fmtop}]{
    FIGS/figures9_4.pdf
}\captionof{figure}{ %
    Separation and gene expression of INA-6 subpopulations. \tile{A} Schematic of
    “Well-Plate Sandwich Centrifugation” (WPSC) separating nMA- from MA-INA6. A
    co-culture 96-well plate is turned upside down and attached on top of a
    “catching plate”, forming a “well-plate sandwich”. nMA-INA6 cells are collected
    in the catching plate by subsequent rounds of centrifugation and gentle washing.
    MA-INA6 are enzymatically dissociated from hMSCs or by rough pipetting.
    Subsequent RNAseq of MSC-interacting subpopulations reveals distinct expression
    clusters [right, multidimensional scaling plot (MDS) ($n=5$)]. \tile{B}
    Separation was microscopically tracked after each centrifugation step.
    \tile{C-E} RT-qPCR of genes derived from RNAseq results. Expression was
    normalized to the median of CM-INA6. Samples include those used for RNAseq and
    six further co-cultures ($n=11$; non-detects were discarded). \tile{C} Adhesion
    factors, ECM proteins, and matrix metalloproteinases. \tile{D} Factors involved
    in bone remodeling and bone homing chemokines. \tile{E} Factors involved in
    (immune) signaling. \tile{Statistics} (C-E): Paired t-test. Datapoints represent the
    mean of three (B-E) technical replicates. INA-6 were isolated from independent
    co-cultures with hMSCs from five (A, B), nine (C-E) unique donors.
}\label{fig:4} % > Is referenced in supplemental



Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam



% ## Fig. 5
\includeimage[.97][{\fmleft} {\fmbottom} {\fmright} {\fmtop}]{
    FIGS/figures9_5.pdf
}\captionof{figure}{ %
    Functional analysis of MSC-interacting subpopulations \tile{(A-C)}
    Functional enrichment analysis of differentially expressed genes (from
    RNAseq) using Metascape. \tile{A} Gene ontology (GO) cluster analysis of
    gene lists that are unique for MA (left) or nMA (right) INA-6. Circle nodes
    represent subsets of input genes falling into similar GO-term. Node size
    grows with the number of input genes. Node color defines a shared parent
    GO-term. Two nodes with a $\text{similarity score} > 0.3$ are linked. \tile{B}
    Enrichment analysis of pairwise comparisons between MA subpopulations and
    their overlaps (arranged in columns). GO terms were manually picked and
    categorized (arranged in rows). Raw Metascape results are shown in
    \refapdx{apdx:supplemental}{fig:S6}. For each GO-term, the p-values (x-axis) and the
    counts of matching input genes (circle size) were plotted. The lowest row
    shows enrichment of gene lists from the TRRUST-database. \tile{C} Circos
    plots by Metascape. Sections of a circle represent lists of differentially
    expressed genes. Purple lines connect same genes appearing in two gene
    lists. \(\cap\): Overlapping groups, MA: MSC-adhering, nMA:
    non-MSC-adhering, CM: MSC-Conditioned Medium. \tile{D} INA-6 were
    co-cultured on confluent hMSC for \SI{24}{\hour} or \SI{48}{\hour},
    separated by WPSC and sub-cultured for \SI{48}{\hour} under IL-6 withdrawal
    ($n=6$), except the control (IL-6 + INA-6) ($n=3$). Signals were
    normalized (red line) to INA-6 cells grown without hMSCs and IL-6 ($n=3$).
    \tile{Statistics} (D): Paired t-test, two-factor RM-ANOVA. Datapoints represent the
    mean of four technical replicates. INA-6 were isolated from independent
    co-cultures with hMSCs from six unique donors. }\label{fig:5}


Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% ## Fig. 6
\includeimage[1][{\fmleft} 3.3in {\fmright} {\fmtop}]{ FIGS/figures9_6.pdf }
\splitcaption{%     
    Survival of patients with multiple myeloma regarding the expression levels
    of adhesion and bone retention genes. \tile{A} p-value distribution of genes
    associated with patient survival ($n=535$) depending on high or low
    expression levels. Red dashed line marks the significance threshold of
    $p\text{-adj}=0.05$. Histogram of $p$-values was plotted using a bin width
    of $-\log_{10}(0.05)/2$. Patients with high and low gene expression were
    delineated using maximally selected rank statistics (maxstat). \tile{B}
    Survival curves for three genes taken from the list of adhesion genes shown
    in (A), maxstat thresholds defining high and low expression were:
    \textit{CXCL12}: 81.08; \textit{DCN}: 0.75; \textit{TGM2}:
    \SI{0.66}{\normcounts}.
    \tile{C} Gene expression (RNAseq, $n=873$) measured
    in normalized counts (edgeR) of \textit{CXCL12}, \textit{DCN} in Bone Marrow
    Plasma Cell (BMPC), Monoclonal Gammopathy of Undetermined Significance
    (MGUS), smoldering Multiple Myeloma (sMM), Multiple Myeloma (MM), Multiple
}{%
    Myeloma Relapse (MMR), Human Myeloma Cell Lines (HMCL). The red dashed line
    marks one normalized read count. \tile{Statistics} (A, B): Log-rank test;
    (C): Kruskal-Wallis, Mann–Whitney U Test. All $p$-values were corrected
    using the Benjamini-Hochberg procedure.
}\label{fig:6}




Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

    % ## Tab. 1
    {
        \footnotesize
        \newcommand{\myheader}{
            \hline
            \textbf{Regulation during disease progression} & \textbf{Gene} & \textbf{Ensemble ID} & \textbf{Progression Free / Overall Survival} & \textbf{Better Prognosis with high/low expression} & \multicolumn{2}{p{3cm}|}{\textbf{Association of expression with survival}}                    \\
            \hhline{~~~~~--}
            &               &                      &                                              &                                                    & \textbf{[p-unc]}                                                      & \textbf{[p-adj]} \\
            \hline
        }

        \begin{longtable}{|>{\bfseries}p{3cm}|>{\bfseries}p{1.9cm}|p{3cm}|p{2cm}|p{2cm}|p{1.5cm}|p{1.5cm}|}
            \caption{Adhesion and ECM genes (shown in \autoref{fig:6}A) were
                filtered by their association with patient survival (p-adj. < 0.01)
                and were categorized as continuously downregulated during disease
                progression. The complete list is presented in
                \refapdx{apdx:supplemental}{tab:S2}. Bone Marrow Plasma Cells
                (BMPC), Monoclonal Gammopathy of Undetermined Significance (MGUS),
                smoldering Multiple Myeloma (sMM), Multiple Myeloma (MM), and
                Multiple Myeloma Relapse (MMR). p-unc: unadjusted p-values; p-adj:
                p-values adjusted using the Benjamini-Hochberg method with 101
            genes.}\label{tab:1}                                                  \\
            \myheader
            \endfirsthead

            \longtablecaptions{3}{}

            \hline

            \multirowcell{3}{3cm}{Not Downregulated (or overall low expression)}
             & CCNE2  & ENSG00000175305 & Overall    & low  & 5.34E-04 & 8.64E-03 \\
            \hhline{~------}
             & MMP2   & ENSG00000087245 & Prog. Free & high & 2.29E-05 & 2.32E-03 \\
            \hhline{~------}
             & OSMR   & ENSG00000145623 & Prog. Free & high & 5.67E-04 & 7.15E-03 \\
            \hhline{~------}
            \hline

            \multirowcell{8}{2.7cm}{Continuously Downregulated (BMPC > MGUS > sMM > MM > MMR)}
             & AXL    & ENSG00000167601 & Overall    & high & 3.64E-05 & 1.84E-03 \\
            \hhline{~------}
             & COL1A1 & ENSG00000108821 & Prog. Free & high & 3.03E-04 & 4.37E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 5.93E-04 & 8.64E-03 \\
            \hhline{~------}
             & CXCL12 & ENSG00000107562 & Prog. Free & high & 1.16E-04 & 2.93E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 6.48E-04 & 8.64E-03 \\
            \hhline{~------}
             & CYP1B1 & ENSG00000138061 & Overall    & high & 6.84E-04 & 8.64E-03 \\
            \hhline{~------}
             & DCN    & ENSG00000011465 & Overall    & high & 2.47E-04 & 8.33E-03 \\
            \hhline{~------}
             & LRP1   & ENSG00000123384 & Overall    & high & 4.34E-04 & 8.64E-03 \\
            \hhline{~------}
             & LTBP2  & ENSG00000119681 & Prog. Free & high & 9.03E-05 & 2.93E-03 \\
            \hhline{~------}
             & CYP1B1 & ENSG00000138061 & Overall    & high & 6.84E-04 & 8.64E-03 \\
            \hhline{~------}
             & DCN    & ENSG00000011465 & Overall    & high & 2.47E-04 & 8.33E-03 \\
            \hhline{~------}
             & LRP1   & ENSG00000123384 & Overall    & high & 4.34E-04 & 8.64E-03 \\
            \hhline{~------}
             & LTBP2  & ENSG00000119681 & Prog. Free & high & 9.03E-05 & 2.93E-03 \\
            \hhline{~------}
             & MFAP5  & ENSG00000197614 & Prog. Free & high & 2.43E-04 & 4.09E-03 \\
            \hhline{~------}
             & MMP14  & ENSG00000157227 & Prog. Free & high & 6.93E-05 & 2.93E-03 \\
            \hhline{~------}
             & MYL9   & ENSG00000101335 & Prog. Free & high & 1.46E-04 & 2.95E-03 \\
            \hhline{~~~----}
             &        &                 & Overall    & high & 1.56E-05 & 1.57E-03 \\
            \hline
        \end{longtable}
    }


Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% ## Fig. 7
\includeimage[1][{\fmleft} 6.8in {\fmright} {\fmtop}]{
    FIGS/figures9_7.pdf
}\captionof{figure}{
    Proposed model of “Detached Daughter Driven Dissemination” (DDDD) in
    aggregating multiple myeloma. \tile{Heterotypic Interaction} Malignant
    plasma cells colonize the bone marrow microenvironment by adhering to an MSC
    (or osteoblast, ECM, etc.) to maximize growth and survival through paracrine
    and adhesion mediated signaling, even if contact may trigger initial
    apoptosis. Gene expression will focus on establishing a strong anchor within
    the bone marrow, but also on attracting other myeloma cells (via secretion
    of ECM factors and CXCL12/CXCL8, respectively). \tile{Cell Division} Cell
    fission can generate one daughter cell that no longer adheres to the MSC
    (nMA). \tile{Homotypic Interaction} If myeloma cells have the capacity to
    grow as aggregates, the daughter cell stays attached to their MSC-adhering
    mother cell (MA). \tile{Re-Adhesion} The daughter cell “rolls around” the
    mother cell until it re-adheres to the MSC. Our model estimates the rolling
    duration to be \SIrange{1}{10}{\hour} long. \tile{Proliferation \& Saturation} We
    estimate that a single myeloma cell covers one MSC completely after roughly
    four population doublings. When heterotypic adhesion is saturated,
    subsequent daughter cells benefit from a homotypic interaction, since they
    stay close to growth-factor secreting MSCs and focus gene expression on
    proliferation (e.g. driven by E2F) and not adhesion (driven by NF-κB).
    \tile{Critical Size} Homotypic interaction is weaker than heterotypic
    interaction, and each cell fission destabilizes the aggregate. Hence,
    detachment of myeloma cells may depend mostly on aggregate size.
    \tile{Dissemination} After myeloma cells have detached, they gained a
    viability advantage through IL-6-independence (with unknown duration), which
    enhances their survival outside of the bone marrow and allows them to spread
    throughout the body. }\label{fig:7}



Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet

% lorem ipsum

% V-Well is described in \refapdx{apdx:supplemental}{fig:S1}

% V-Well is described in \refapdx{apdx:supplemental}{fig:S1}, \ref{fig:S2}

% % Image cytometry is described in \autoref{subapdx:sup_figtabs} \autoref{fig:S2}

% Full analysis is shown in \refapdx{subapdx:example_analysis}

% % ======================================================================
% % == Discussion
% % ======================================================================
\unnsubsection{Discussion}\label{C1:discussion} % > Require unique label
\ %
lorem ipsum

% % == Paper 1 ===========================================================
% % > You could import .pdf here, but chapter based theses should apply the 
% % > manuscripts into the formatting of the thesis
% % \addpdf{Research Article: Cancer Research Communications}{PUBLICATIONS/AACR.pdf}





