
\unnsection{Summarising Discussion}


% ======================================================================
% From Paper 1 (AACR):

% In this study, we developed an in vitro model to investigate the
% attachment/detachment dynamics of INA-6 cells to/from hMSCs and
% established methods to isolate the attached and detached intermediates
% nMA-INA6 and MA-INA6. Second, we characterized a cycle of
% (re)attachment, division, and detachment, linking cell division to the
% switch that causes myeloma cells to detach from hMSC adhesion (Fig. 7).
% Thirdly, we identified clinically relevant genes associated with patient
% survival, in which better or worse survival was based on the adherence
% status of INA­6 to hMSCs.

% INA-6 cells emerged as a robust choice for studying myeloma
% dissemination in vitro, showing rapid and strong adherence, as well as
% aggregation exceeding MSC saturation. The IL-6 dependency of INA-6
% enhanced the resemblance of myeloma cell lines to patient samples, with
% INA-6 ranking 13th among 66 cell lines (46). Despite variations in bone
% marrow MSCs between multiple myeloma (MM) and healthy states, we
% anticipated the robustness of our results, given the persistent strong
% adherence and growth signaling from MSCs to INA-6 during co-cultures
% (47).

% We acknowledge that INA-6 cells alone cannot fully represent the
% complexity of myeloma aggregation and detachment dynamics. However, the
% diverse adhesive properties of myeloma cell lines pose a challenge. We
% reasoned that attempting to capture this complexity within a single
% publication would not be possible. Our focus on INA-6 interactions with
% hMSCs allowed for a detailed exploration of the observed phenomena, such
% as the unique aggregation capabilities that facilitate the easy
% detection of detaching cells in vitro. The validity of our data was
% demonstrated by matching the in vitro findings with the gene expression
% and survival data of the patients (e.g. CXCL12, DCN, and TGM2
% expression, n=873), ensuring biological consistency and generalizability
% regardless of the cell line used. 

% The protocols presented in this study offer a cost-efficient and
% convenient solution, making them potentially valuable for a broader
% study of cell interactions. We encourage optimizations to meet the
% varied adhesive properties of the samples, such as decreasing the number
% of washing steps if the adhesive strength is low. We caution against
% strategies that average over multiple cell lines without prior
% understanding their diverse attachment/detachment dynamics, such as
% homotypic aggregation. Such detailed insights may prove instrumental
% when considering the diversity of myeloma patient samples across
% different disease stages (34,35).

% The intermediates, nMA-INA6 and MA-INA6, were distinct but shared
% similarities in response to cell stress, intrinsic apoptosis, and
% regulation by p53. Unique regulatory patterns were related to central
% transcription factors: E2F1 for nMA-INA6; and NF-κB, SRF, and JUN for
% MA-INA6. This distinction may have been established through antagonism
% between p53 and the NF-κB subunit RELA/p65 (38,39). Similar regulatory
% patterns were found in transwell experiments with RPMI1-8226 myeloma
% cells, where direct contact with the MSC cell line HS5 led to NF-κB
% signaling and soluble factors to E2F signaling (20).

% The first subpopulation, nMA-INA6, represented proliferative and
% disseminative cells; nMA-INA6 drove detachment through cell division,
% which was regulated by E2F, p53, and likely their crosstalk (48). They
% upregulate cell cycle progression genes associated with worse prognosis,
% because proliferation is a general risk factor for an aggressive disease
% course (49). Additionally, nMA-INA6 survived IL-6 withdrawal better than
% CM-INA6 and MA-INA6, implying their ability to proliferate independently
% of the bone marrow (2). Indeed, xenografted INA-6 cells developed
% autocrine IL-6 signaling but remained IL-6-dependent after explantation
% (24). The increased autonomy of nMA-INA-6 cells can be explained by the
% upregulation of IGF-1, being the major growth factor for myeloma cell
% lines (43). Other reports characterized disseminating cells differently:
% Unlike nMA-INA6, circulating myeloma tumor cells were reported to be
% non-proliferative and bone marrow retentive (50). In contrast to
% circulating myeloma tumor cells, nMA-INA6 were isolated shortly after
% detachment and therefore these cells are not representative of further
% steps of dissemination, such as intravasation, circulation or
% intravascular arrest (3). Furthermore, Brandl et al. described
% proliferative and disseminative myeloma cells as separate entities,
% depending on the surface expression of CD138 or JAM-C (4,51). Although
% CD138 was not differentially regulated in nMA-INA6 or MA-INA6, both
% subpopulations upregulated JAM-C, indicating disease progression (51). 

% Furthermore, nMA-INA6 showed that cell division directly contributed to
% dissemination. This was because INA-6 daughter cells emerged from the
% mother cell with distance to the hMSC plane in the 2D setup. A similar
% mechanism was described in an intravasation model in which tumor cells
% disrupt the vessel endothelium through cell division and detach into
% blood circulation (52). Overall, cell division offers key mechanistic
% insights into dissemination and metastasis.

% The other subpopulation, MA-INA6, represented cells retained in the bone
% marrow; MA-INA6 strongly adhered to MSCs, showed NF-κB signaling, and
% upregulated several retention, adhesion, and ECM factors. The production
% of ECM-associated factors has recently been described in MM.1S and
% RPMI-8226 myeloma cells (53). Another report did not identify the
% upregulation of such factors after direct contact with the MSC cell line
% HS5; hence, primary hMSCs may be crucial for studying myeloma-MSC
% interactions (20). Moreover, MA-INA6 upregulated adhesion genes
% associated with prolonged patient survival and showed decreased
% expression in relapsed myeloma. As myeloma progression implies the
% independence of myeloma cells from the bone marrow (2,46), we
% interpreted these adhesion genes as mediators of bone marrow retention,
% decreasing the risk for dissemination and thereby potentially prolonging
% patient survival. However, the overall impact of cell adhesion and ECM
% on patient survival remains unclear. Several adhesion factors have been
% proposed as potential therapeutic targets (51,54). Recent studies have
% described the prognostic value of multiple ECM genes, such as those
% driven by NOTCH (53). Another study focused on ECM gene families, of
% which only six of the 26 genes overlapped with our gene set (Tab. S2)
% (55). The expression of only one gene (COL4A1) showed a different
% association with overall survival than that in our cohort. The lack of
% overlap and differences can be explained by dissimilar definitions of
% gene sets (homology vs. gene ontology), methodological discrepancies,
% and cohort composition.

% In summary, our in vitro model provides a starting point for
% understanding the initiation of dissemination and its implications for
% patient survival, providing innovative methods, mechanistic insights
% into attachment/detachment, and a set of clinically relevant genes that
% play a role in bone marrow retention. These results and methods might
% prove useful when facing the heterogeneity of disseminative behaviors
% among myeloma cell lines and primary materials.



\unnsubsection{Time Lapse}
Lorem Ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
sed diam voluptua. At vero eos et accusam et justo duo dolores et ea
rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem
ipsum

\unnsubsection{Myeloma}
Lorem Ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit
amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
sed diam voluptua. At vero eos et accusam et justo duo dolores et ea
rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem
ipsum

\unnsubsection{Semi-Automated Analysis Improves Agility During Establishing new \textit{in vitro} Methods}
Was plotastic useful for me?
- Yes incredibly. I was able to perform the statistical tests and
visualize the data in a fraction of the time that I would have needed
manually. This allowed me to focus on the interpretation of the results
and the writing of the manuscript.
There was one particular example where my analysis was so fast, that I
fed raw datatables during microscopy into python scripts and was able to
adapt the experimental technique during the experiment. This allows for
an agile and adaptive work environment that is not possible with manual
analysis and proved invaluable during development of  \textit{in
    vitro} methods.
- These experiments benefited from the use of plotastic, as the

Further research is needed to assess the true impact of semi-automated
analysis on the agility of establishing new \textit{in vitro} methods.


\unnsubsection{Plotastic and Jupyter Notebooks as a Standard for Research Documentation}
\label{dis:plotastic_jupyter}


\unnsubsection{How Biosciences can Benefit from Standards of Software Development}
\label{dis:code_quality}
%%%%%%%
chapter 2 has laid down the benefits of using plotastic in combination with jupyter notebooks


It is important to note that  GitHub are not reserved for software development.
Bioscientists are encouraged to use these tools as well which are available with
free, robust well-designed and intuitive graphical user interfaces developed by
the distinguished experts of the open source community. Its robustness is
unparalleled and have withstood the test of time.

It is a database that documents every minor change thoroughly, noting every little
change in text, providing unmatched flexibility but also replicability.
Researchers can 

GitHub encourages the use by Scientists.




% \unnsubsection{Transfer Effects of Coding on Bioscientific Research}



% Finally, the author makes the argument that basic coding skills not only allows
% for efficient usage of common software tools, but could also enhance the
% interaction with computers in general. Intriguingly, there is a long-lasting
% discussion if coding has benefits on overall cognitive
% capability~\cite{schererEvidenceCognitiveBenefits2021,schererTechnologyMindDoes2018}.
% These so-called \textit{``transfer effects''} are similar to those surrounding
% e.g. chess, Latin, video gaming and brain training.
% \cite{salaDoesFarTransfer2017}, assuming that learning to code also improves
% problem-solving in other areas, such as mathematics or science. Although the
% author does not have a stand in this discussion, it is reasonable to state that
% coding improves a general understanding of computers and software tools. Since
% most software tools in the biosciences are black boxes with intimidating
% complexity. Still, when confronted with issues, researchers must determine if
% the error is on their side or the software's in order to assess possible
% troubleshooting strategies. It is reasonable to argue that a good understanding
% of the software is crucial to facilitate that decision-making process.

% a solid understanding of computers facilitates the decision-making process when
% confronted with issues, allowing researchers to determine if the error is on
% their side or the software's. This is only possible if the researcher has a good
% understanding of the software.


% Learning to code has been hypothesized to have generalizable benefits for
% cognitive function and problem
% solving~\cite{schererEvidenceCognitiveBenefits2021}. Here the author reflects on
% his own experience with coding and how it has benefitted his research beyond the
% development of plotastic

% Coding skills enhance a researcher's understanding of the software they use.
% This understanding allows them to decipher the underlying algorithms and
% assumptions of the software. Complex proprietary software like Zen, Imaris,
% FlowJo, etc., often operate as black boxes. When confronted with issues,
% researchers must determine if the error is on their side or the software's. This
% decision-making process is facilitated by a good understanding of the software.
% Basic coding skills also open the door to open-source software, which offers
% state-of-the-art analysis techniques not available in proprietary software.

% - In general, coding skills help improve a researchers general understanding of
% every software he uses. This is because the researcher can understand the
% underlying algorithms and assumptions of the software. Complex proprietary
% software to use state-of-the-art equipment (Zen, Imaris, FlowJo, etc.) are often
% black boxes. The researcher will be confronted with issues and has to make
% constant decisions if the error is on his side, or on the software side in order
% to think of feasable troubleshooting strategies. This is only possible if the
% researcher has a good understanding of the software. A basic understanding of
% code also opens the door to open-source software, allowing for state-of-the-art
% analysis techniques that are not available in proprietary software, but their
% installation and usage is often overestimated by researchers without coding
% skills, while coding skills allow for adaptations of this software to the
% specific needs of the researcher.




In conclusion, as bioscience continues to generate increasingly complex datasets,
the distinction between biologists and bioinformaticians blurs, emphasizing the
need for all researchers to adopt computational tools. The development of new
software to handle semi-big data effectively is not just an enhancement but a
necessity to ensure the future reliability and efficiency of scientific
research. This thesis proposes a framework for understanding and addressing the
semi-big data challenges, setting the stage for a discussion on innovative
solutions like the software tools described in subsequent chapters.

In conclusion, the integration of coding in bioscience research is not just a
trend but a necessity. As the field continues to evolve, the demarcation between
biologists and computational scientists blurs, underscoring the importance of
coding skills for the next generation of researchers. The ability to code is
fast becoming an indispensable asset, as integral to bioscience as traditional
laboratory skills.


The reproducibility of qPCR has been a long-standing issue in the field%
\footnotequote{
    By 2009 it was pretty obvious that many publications using PCR based methods were seriously flawed
}{bustinReproducibilityBiomedicalResearch2014}.