



% ======================================================================
% From Paper 1 (AACR):

% In this study, we developed an in vitro model to investigate the
% attachment/detachment dynamics of INA-6 cells to/from hMSCs and
% established methods to isolate the attached and detached intermediates
% nMA-INA6 and MA-INA6. Second, we characterized a cycle of
% (re)attachment, division, and detachment, linking cell division to the
% switch that causes myeloma cells to detach from hMSC adhesion (Fig. 7).
% Thirdly, we identified clinically relevant genes associated with patient
% survival, in which better or worse survival was based on the adherence
% status of INA­6 to hMSCs.

% INA-6 cells emerged as a robust choice for studying myeloma
% dissemination in vitro, showing rapid and strong adherence, as well as
% aggregation exceeding MSC saturation. The IL-6 dependency of INA-6
% enhanced the resemblance of myeloma cell lines to patient samples, with
% INA-6 ranking 13th among 66 cell lines (46). Despite variations in bone
% marrow MSCs between multiple myeloma (MM) and healthy states, we
% anticipated the robustness of our results, given the persistent strong
% adherence and growth signaling from MSCs to INA-6 during co-cultures
% (47).

% We acknowledge that INA-6 cells alone cannot fully represent the
% complexity of myeloma aggregation and detachment dynamics. However, the
% diverse adhesive properties of myeloma cell lines pose a challenge. We
% reasoned that attempting to capture this complexity within a single
% publication would not be possible. Our focus on INA-6 interactions with
% hMSCs allowed for a detailed exploration of the observed phenomena, such
% as the unique aggregation capabilities that facilitate the easy
% detection of detaching cells in vitro. The validity of our data was
% demonstrated by matching the in vitro findings with the gene expression
% and survival data of the patients (e.g. CXCL12, DCN, and TGM2
% expression, n=873), ensuring biological consistency and generalizability
% regardless of the cell line used. 

% The protocols presented in this study offer a cost-efficient and
% convenient solution, making them potentially valuable for a broader
% study of cell interactions. We encourage optimizations to meet the
% varied adhesive properties of the samples, such as decreasing the number
% of washing steps if the adhesive strength is low. We caution against
% strategies that average over multiple cell lines without prior
% understanding their diverse attachment/detachment dynamics, such as
% homotypic aggregation. Such detailed insights may prove instrumental
% when considering the diversity of myeloma patient samples across
% different disease stages (34,35).

% The intermediates, nMA-INA6 and MA-INA6, were distinct but shared
% similarities in response to cell stress, intrinsic apoptosis, and
% regulation by p53. Unique regulatory patterns were related to central
% transcription factors: E2F1 for nMA-INA6; and NF-κB, SRF, and JUN for
% MA-INA6. This distinction may have been established through antagonism
% between p53 and the NF-κB subunit RELA/p65 (38,39). Similar regulatory
% patterns were found in transwell experiments with RPMI1-8226 myeloma
% cells, where direct contact with the MSC cell line HS5 led to NF-κB
% signaling and soluble factors to E2F signaling (20).

% The first subpopulation, nMA-INA6, represented proliferative and
% disseminative cells; nMA-INA6 drove detachment through cell division,
% which was regulated by E2F, p53, and likely their crosstalk (48). They
% upregulate cell cycle progression genes associated with worse prognosis,
% because proliferation is a general risk factor for an aggressive disease
% course (49). Additionally, nMA-INA6 survived IL-6 withdrawal better than
% CM-INA6 and MA-INA6, implying their ability to proliferate independently
% of the bone marrow (2). Indeed, xenografted INA-6 cells developed
% autocrine IL-6 signaling but remained IL-6-dependent after explantation
% (24). The increased autonomy of nMA-INA-6 cells can be explained by the
% upregulation of IGF-1, being the major growth factor for myeloma cell
% lines (43). Other reports characterized disseminating cells differently:
% Unlike nMA-INA6, circulating myeloma tumor cells were reported to be
% non-proliferative and bone marrow retentive (50). In contrast to
% circulating myeloma tumor cells, nMA-INA6 were isolated shortly after
% detachment and therefore these cells are not representative of further
% steps of dissemination, such as intravasation, circulation or
% intravascular arrest (3). Furthermore, Brandl et al. described
% proliferative and disseminative myeloma cells as separate entities,
% depending on the surface expression of CD138 or JAM-C (4,51). Although
% CD138 was not differentially regulated in nMA-INA6 or MA-INA6, both
% subpopulations upregulated JAM-C, indicating disease progression (51). 

% Furthermore, nMA-INA6 showed that cell division directly contributed to
% dissemination. This was because INA-6 daughter cells emerged from the
% mother cell with distance to the hMSC plane in the 2D setup. A similar
% mechanism was described in an intravasation model in which tumor cells
% disrupt the vessel endothelium through cell division and detach into
% blood circulation (52). Overall, cell division offers key mechanistic
% insights into dissemination and metastasis.

% The other subpopulation, MA-INA6, represented cells retained in the bone
% marrow; MA-INA6 strongly adhered to MSCs, showed NF-κB signaling, and
% upregulated several retention, adhesion, and ECM factors. The production
% of ECM-associated factors has recently been described in MM.1S and
% RPMI-8226 myeloma cells (53). Another report did not identify the
% upregulation of such factors after direct contact with the MSC cell line
% HS5; hence, primary hMSCs may be crucial for studying myeloma-MSC
% interactions (20). Moreover, MA-INA6 upregulated adhesion genes
% associated with prolonged patient survival and showed decreased
% expression in relapsed myeloma. As myeloma progression implies the
% independence of myeloma cells from the bone marrow (2,46), we
% interpreted these adhesion genes as mediators of bone marrow retention,
% decreasing the risk for dissemination and thereby potentially prolonging
% patient survival. However, the overall impact of cell adhesion and ECM
% on patient survival remains unclear. Several adhesion factors have been
% proposed as potential therapeutic targets (51,54). Recent studies have
% described the prognostic value of multiple ECM genes, such as those
% driven by NOTCH (53). Another study focused on ECM gene families, of
% which only six of the 26 genes overlapped with our gene set (Tab. S2)
% (55). The expression of only one gene (COL4A1) showed a different
% association with overall survival than that in our cohort. The lack of
% overlap and differences can be explained by dissimilar definitions of
% gene sets (homology vs. gene ontology), methodological discrepancies,
% and cohort composition.

% In summary, our in vitro model provides a starting point for
% understanding the initiation of dissemination and its implications for
% patient survival, providing innovative methods, mechanistic insights
% into attachment/detachment, and a set of clinically relevant genes that
% play a role in bone marrow retention. These results and methods might
% prove useful when facing the heterogeneity of disseminative behaviors
% among myeloma cell lines and primary materials.


\unnsection{Summarising Discussion}%
\label{sec:summarising_discussion}%
% 
% ======================================================================
\unnsubsection{Potentials and Challenges of Live-Cell Imaging for Exploratory Cell Biology}%
\label{sec:discussion_potential_breakthroughs}%
Exploratory experimentation emphasizes discovering and characterizing novel
phenomena \cite{mattigClassifyingExploratoryExperimentation2022}. Exploratory
cell biology often leverages emerging technologies to visualize and analyze the
mechanisms of cell behavior dynamically. Such approaches allow real-time
observations that can lead to unexpected insights and breakthroughs. In this
project, the application of live-cell imaging proved pivotal.

\textbf{Direct Observation of Complexity and Novelty:}
Initially, the project did not focus on \textit{in vitro} myeloma cell
dissemination. The project's research focus shifted when making the unexpected
\dashedsentence{or argueably insignificant} observation of cancer cells
detaching from aggregates. This shows the transformative power of time-lapse
microscopy or live cell imaging \cite{coleLivecellImaging2014}. For the author,
live-cell imaging provides an observation method that's unmatched in intuition
and directness. Unlike RNA sequencing, which can obscure biological processes
behind cryptic data, live-cell imaging offers a clear view into the dynamic
cellular events as they unfold.

Such clarity was particularly effective in revealing the detachment of cells
following division, a phenomenon that might be overlooked in static analyses.
Multiple parameters can be read out in parallel, such as both time and aggregate
size for detachments to begin. Also, complex cellular behavior can be deduced
from movement, or rather lack thereof, which was interpreted as re-attachment of
\INA daughter cells to the \ac{hMSC} monolayer. This allowed for measuring the
duration of \nMAina existing until re-attaching and turning into \MAina. This
information was helpful when designing experiments to prove that dissemination
is initiated by cell division, requiring precise timing to capture the detached
daughter cells right after cell division. Together, live cell imaging enabled
key mechanistic insights in understanding the dynamics involved in multicellular
interactions by integrating the study of multiple phenomena at once.

\textbf{\textit{The cells are doing ...that thing again!}} Exploring video data
begins with the search of scientific novelties. In order to correctly identify
cellular phenomena relevant to the research question, a deep
understanding of cell biology is required, e.g. in field of cell dynamics to
read migratory behavior \cite{nalbantExploratoryCellDynamics2018}. This is a
challenge for both students and experienced researchers, since finding the
academically correct terms to describe observations is difficult, especially for
novel phenomena or a sequence of events that can overlap. After all, cell
biology is taught using textbooks, not videos. For this project in particular,
the used terminology was revised frequently, being caused by the constant
struggle of finding the middle-ground between the precice description of
observations, the compatibility with results from other experiments,
comprehensability, and memorability. Ultimately, comprehensability and
memorability were prioritized to maximize adoption of the new terminology by
other researchers. For instance, \emph{non MSC adherence} was chosen over
\emph{mobile interaction}, \emph{aggregation} over \emph{homotypic interaction},
and \emph{detachment event} over \emph{in vitro metastasis}. In general, the gap
between observations and their description remains a challenge in exploratory
cell biology that might be overlooked. This gap could be bridged by currently
available multimodal \acp{LLM} like \texttt{ChatGPT-4o}: These models could
match recorded phenomena with descriptions and images that were amassed in the
literature over decades. By doing so, researchers not only use established
terminology instead of inventing new terms, but also minimize the risk of
missing potential discoveries.

\textbf{Why Hide Videos Behind a Download Link?} A major challenge remains in how
to effectively present these dynamic observations in a publishable format, as
traditional scientific publications and websites are not equipped to display
video data. Instead, it is common practice to assemble video frames into static
figures, presumably to support both online and printed reading habits
\cite{perasDigitalPaperReading2023}. Representative example videos are then
relegated to supplementary data. Although supplementary data is downloaded
often, most biomedical researchers favor a presentation of additional figures
and tables directly on the journal's website
\cite{priceRoleSupplementaryMaterial2018}. Given the increasing availability of
video data\footnote{The number of \texttt{PubMed} articles with "live cell
    imaging" doubled from 2011 to 2023.}, embedding video content next to figures
and tables on the article's website does make a compelling case. In fact, the
journal \emph{Nature} does offer this feature already, but rarely used
\cite{NatureVideoContent}. In the end, there is no reason to not present videos
alongsife figures and tables, as they can be as informative, and potentially
more so. Such new standards can benefit other fields of medicine, as videos
provide the best medium for first aid, medical emergency and education
\cite{guptaDatasetMedicalInstructional2023}.

Overall, Live-cell imaging has proven indispensable in exploratory cell biology,
uncovering dynamic cellular phenomena that static analyses often miss. This
technique shifted the project’s focus by revealing unexpected cell behaviors,
like detachment during division, emphasizing the need for integrating real-time
observations with molecular data. By making such dynamic processes visible,
live-cell imaging not only enriches our understanding but also challenges us
to enhance how scientific findings are presented, advocating for greater
accessibility of video data in scientific publications.


% ======================================================================
\unnsubsection{Challenges in Quantifying Microscopy Data}%
\label{sec:discussion_quantifying_microscopy}%
Quantifying microscopy data is critical for both analytic and exploratory
approaches to microscopy: For instance, microscopic assessment of live/dead
cells should produce bar charts presenting cell viabilities
\cite{spaepenDigitalImageProcessing2011}, whereas describing novel phenomena
should be supported by charts proving the reproducibility of claimed
observations. Microscopy data is source of vast amount and types of information:
cell morphology; organelle count, shape, and distribution; membrane and lipid
distribution; protein localization, DNA content, et cetera. However, leveraging
this information has always been limited by the ability to extract quantitative
data from images \cite{galbraithPumpingVolume2023}. This led to many advances
spanning multiple fields of biology, image processing and machine learning,
potentially ushering a new era of image cytometry
\cite{guptaDeepLearningImage2019}.



\textbf{Technical Considerations for Acquiring Accurate Image Data:}
In order to capture rare cellular events with a frequency sufficient for
statistical analysis, this study chose high temporal resolution and spatial
depth: We utilized \SI{1}{frame} every \SI{15}{\minute}, suitable for tracking
cell migration \cite{huthSignificantlyImprovedPrecision2010}, but too slow for
intricate movements or intracellular processes. Spatial resolution is a
compromise between detail and the total observed surface area. We favored the
latter to allow the exploration of potentially rare events, and acquired a
\dashedsentence{somewhat arbitrarily} large surface area of up to
\SI{13}{\milli\meter\squared}. Ultimately, we assessed only approx. a quarter of
the acuired surface area, as that was sufficient to gather enough events for
each time bin. Such extensive automated video acquisition poses high demands on
microscopy equipment, including an incubation setup and motorized stage top. The
total size of video files can also complicate storage, transfer and analysis.
The raw video data from chapter 1 comprises \SI{80}{GB}
\cite{biostudiesBioStudiesEuropeanBioinformatics}; however, far more data was
acquired due to protocol optimizations and treatments not shown in this work.
File size could have been reduced by acquiring in an 8-bit image format,
although a larger bit-depth could be necessary for precise and/or sensitive
fluorescence microscopy. Minimizing the acquired surface area could have reduced
file size as well, however the meniskus of the medium led to significant shading
effects that complicated the choice of the surface area for phase contrasting.
Also, archiving large surface scans allows for the search of very rare events in
the course of future projects. After all, HDD space is cheap, while re-acquiring
data is not. Hence, exploratory live cell imaging benefits from settings that
are higher-than-required, if raw data is properly documented and remain
accessible.

\textbf{Manual Analysis for Complex and Unique Phenomena:}
Most Biologists lack the computational expertise to automate analysis of
microscopy data, often reverting to manual analysis. This project also utilized
manual strategies for the detailed characterization of dynamic intercellular
interactions such as attachment, aggregation, detachment, and division. This was
very time-consuming and required a thoughtful categorization strategy and a
disciplined, bias-free execution. However, some analysis tasks are simply unfeasable
for automation. For example, this work
manually counted if two \INA cells interacted homotypically due to coming into
contact with each other, or by staying connected as two daughter cells after
cell division. Automating such a task would require a very sophisticated
algorithm and unfeasable for such unique task.
Hence, manual analysis is unmatched in terms of flexibility and complexity of
categorizations, when compared to computational techniques of image processing.

\textbf{Computational Tools for Analysis of Live-Cell Imaging:}
Despite the benefits of manual analyses, this work would have benefited from
computational automation for the analysis of live-cell imaging. For example, the
task of associating \INA cell detachment with \INA aggregate size and time could
have been automated: Manual analysis consisted of zooming in closely and
watching the time-lapse over and over again until a detachment event was found.
A very tedious task that had to be repeated approx. 50 times for every one of
four independent videos. Instead of manually counting the number of single \INA
cells across time, a pixel segmentation algorithm could have been trained to
detect cells and background. Single cells would be discernable from aggregates
by filtering cells by size. The count of single cells would then be
representative of detached cells, given that the vast majority of INA6 cells
were part of aggregates.

The workload of manual video analysis motivated the purchase of
\texttt{Intellesis}, a software package by \textit{Zeiss} for the \texttt{Zen}
microscopy software ecosystem. \texttt{Intellesis} is a machine learning-based
pixel segmentation software \cite{ZeissOADFeature}. As a feature extractor, it
uses the first convolution layers of the convolutional neural network VGG19
\cite{simonyanVeryDeepConvolutional2015}. \texttt{Intellesis} does not contain a
deep neural network for segmentation, but instead classifies pixel features
using a \emph{random forest classifier}. Random forest is a machine learning
algorithm that \dashedsentence{for small sets of training images} performs
almost as well as deep neural networks, but are computationally far less
demanding \cite{breimanRandomForests2001, richardsonDenseNeuralNetwork2023}. A
comparable hybrid approach was also used by
\citet{qamarHybridCNNRandomForest2023} to segment images of bacterial spores
into eight distinct pixel classes using only 50 training images.


\texttt{Intellesis} proved useful for segmenting single multi-channel images.
However, live cell imaging adds another layer of complexity to image analysis:
The addition of a time axis encodes the motion of image features, a concept that
can be described with the term \emph{optical flow}
\cite{niehorsterOpticFlowHistory2021}. Mathematically speaking, optical flow is
a vector field that describes the motion of image features between consecutive
frames of a video. It can be used to efficiently train machine learning models
on video data \cite{robitailleSelfsupervisedMachineLearning2022}. Without tricks
like optical flow, machine learning algorithms like \texttt{Intellesis} segment
the video frame by frame, ignoring the feature similarities between frames.
This makes segmentation computationally inefficient, but not impossible
\cite{pylvanainenLivecellImagingDeep2023}. 

Together, this project's analses of live-cell imaging data could have benefited
from the use of modern machine learning based tools that have been released in
the past years \cite{pylvanainenLivecellImagingDeep2023}. 




\textbf{Quantitative Potential of Image Cytometry:}

Computational approaches to image analysis are attractive as they remove bias
and increase throughput. However, few biologists have the expertise to develop
and apply image processing pipelines. \textit{Intellesis} integrates well with 
\texttt{Zen}, the software used for image acquisition and is designed to be
accessible to biologists without expertise in image processing, enabling the
conversion of image data into analyzable quantitative outputs.

% facilitated by the machine learning software \textit{Intellesis}, enabled
% high-throughput and precise quantification necessary for tasks like cell cycle
% profiling in fluorescence microscopy. This hybrid approach leveraged
% \textit{Intellesis}’s capabilities to segment and quantify tens of thousands of
% DNA-stained nuclei efficiently, aligning this project with advanced image
% cytometry techniques \cite{guptaDeepLearningImage2019}.


For
fluorescent cell cycle profiling however, an automated approach was necessary to
reach throughput, precision and cell number. For this, \textit{Intellesis} was
used, which is a machine learning-based pixel segmentation software.

\textit{Intellesis} is not an artificial intelligence, as its segmentation algorithm
does not contain a neural network. Instead, it uses a random forest classifier



This approach allowed the automated
extraction of fluorescence brightness for every single nucleus among tens of
thousands of DNA-stained nuclei, followed by plotting the distribution of
DNA-amounts. Using \textit{Intellesis}, this project effectively entered the
field of \emph{image cytometry} \cite{guptaDeepLearningImage2019}, achieving
high levels of automation along with a single-cell precision that's comparable
to flow cytometry (\apdxref{subapdx:figs}{fig:S3}). Although this work has not
applied \textit{Intellesis} to live-cell imaging, this exemplifies the vast
quantitative potential of microscopy \dashedsentence{or rather image cytometry}
in general.



% Here, this project effectively
% entered the field of \emph{image cytometry} \cite{guptaDeepLearningImage2019},
% achieving high levels of automation along with a single-cell precision
% comparable to flow cytometry when performing fluorescent cell cycle profiling.
% This exemplifies the vast quantitative potential of microscopy in general.


% Automated analysis, on the other hand, utilizes pixel segmentation techniques
% either based on sophisticated image-processing pipelines and/or machine
% learning. \textit{Intellesis}, offer solutions by making machine learning based
% pixel segmentation accessible to biologists without expertise in image
% processing, enabling the conversion of video data into analyzable quantitative
% outputs. This approach allows the extraction of meaningful scientific metrics
% from time-lapse experiments, turning dynamic observation into static scientific
% reporting. Using \textit{Intellesis}, this project effectively entered the field
% of \emph{image cytometry} \cite{guptaDeepLearningImage2019}, achieving high
% levels of automation along with a single-cell precision that's comparable to
% flow cytometry when performing fluorescent cell cycle profiling. This
% exemplifies the vast quantitative potential of microscopy in general.


\textbf{Make Intellesis Training Easier and more Robust:}
% either requires know-how of
% image-processing or expensive software. Using expensive software based on machine
% learning assisted 
This project has invested a lot of time in training the machine learning
software \textit{Intellesis} to segment images.

Some image types remain challenging for machine learning-based software
like \textit{Intellesis}, such as

phase contrast images, which have low contrast
between cell edges and the background, making cell separation difficult during
pixel segmentation.

phase
contrast images are more challenging for machine learning based software like
\textit{Intellesis}, since cell edges have low contrast to the background and
single cells are hard to distinguish. This can require extensive annotation of
training images, which can be time-consuming and frustrating. This could be
alleviated by introducing pre-processing steps that emphasize the features of
interest and/or reduce image features that are irrelevant for the analysis. For
example when discerning cell morphology, cell edges can be enhanced using edge
enhancing filters, while noise and irrelevant small details can be removed by
applying a median filter that preserves edges. Both filters are available in
\texttt{Zen}. This approach simplifies the task for machine learning algorithms
and reduces the amount of training data required by avoiding the classifier to
learn irrelevant image features.

% . In summary, the implementation


% applying a median filter to reduce

% e.g. by applying

% intentionally removing details from the images that
% are otherwise extracted as features from the machine learning algorithm, but


% , e.g. by
% applying

% Pre-processing
% images to reduce features extracted by the machine learning algorithm could
% alleviate this problem, for example applying a median filter blurs details while
% preserving edges



% with a median filter could alleviate this problem by blurring details
% while preserving sharp edges, reducing the feature amount

% simplifying the task for machine learning algorithm


% , such as applying blurring details  an edge-preserving median filters, could alleviate
% this problem by simplifying the task for machine learning algorithms and reducing

% This could be alleviated by applying

% require a larger training dataset  of
% \textit{Intellesis}. 

% experienced annotation


% can be more efficient, but it requires
% the generation of proper training data.

% Advances in digital image processing, like \textit{Intellesis}, offer solutions
% by making machine learning accessible to biologists, enabling the conversion of
% video data into analyzable quantitative outputs. This approach allows
% the extraction of meaningful scientific metrics from time-lapse experiments,
% turning dynamic observation into static scientific reporting.


% applying median filters to
% reduce noise, thus simplifying the task for machine learning algorithms during
% post-processing, enhance the analysis without the extensive generation of
% training data.



In summary, the implementation of live-cell imaging in this project not only
enhanced our understanding of myeloma cell behavior but also emphasized the need
for innovative solutions to integrate dynamic cellular data into the rigid
frameworks of scientific communication. The evolution of microscopy and image
analysis into image cytometry continues to push the boundaries of what can be
observed and understood about cell dynamics in disease and health.


\textbf{Lack of Translation Between Machine Learning Experts and Biologists?:}

Current opinions on machine learning speak of training neural networks, without mentioning
random-forest classifiers.
To the author's know


A comparable hybrid approach was also used by \citet{qamarHybridCNNRandomForest2023}
to segment images of bacterial spores
into eight distinct pixel classes using 50 training images.

, which is a very low number compared to the
hundreds of images required for neural networks.




% - Biggest current challenge: How to present observations in a publishable form? After
% all, videos can not be printed into Journals Usage of modern Pixel segmentation
% techniques? In the end, we need something like bar plots to document
% observations by scientific standards! Advances has been made through Intellesis,
% making Machine learning extremely accessible even for biologists, but still,
% phase contrast acquisitions require experienced annotation techniques, or
% generating training data can get very time consuming and even frustrating. Using
% Pre-processing could alleviate this problem, e.g. when only cell morphology is
% required, a median filter can be applied to reduce salt-and-pepper noise while
% preserving edges. This simplifies the task for the machine learning algorithm
% and reduces the amount of training data required. Manual analysis is also a very viable
% option, yet it is very time consuming and requires development of a ver thoughtful
% categorization strategy and disciplined and bias-free application of it. 

\unnsubsection{Conclusion 1: Microscopy On the Brink of Breakthrough, if Image Analysis is Becoming more Available}%
\label{sec:discussion_conclusion_microscopy}%




% ======================================================================
\unnsubsection{Novel Methods of Isolating Adhering Subpopulations}%
\label{sec:discussion_novel_methods}%

In this work, innovative \textit{in vitro} methodologies (Well Plate Sandwich
Centrifugation and V-Well adhesion Assay) were developed. this was required to
fill in gaps of isolating cells with minimized variability introduced by
user-bias to clearly separate subpopulations and precisely quantify them.


It is evident that direct or indirect contact with MM can have different effects
on both hMSCs and Myeloma cells and methods to differentiate between those are
crucial for understanding the change of the \ac{BMME} during \ac{MM} progression
\cite{fairfieldMultipleMyelomaCells2020, dziadowiczBoneMarrowStromaInduced2022}

cite all those methods for cell isolation!
- Turning around wellplates: Doesn't allow isolation, just quantification
- The author did not show all his washing experiments
- Washing is very bad (data not shown): Highly dependent on user:
position of cell on well bottom (border cells receive less force), position of
pipette tip in well (depth, angle and position on bottom)
- This motivated us to explore more reproducible methods

It's a challenge: either quantify cell population, or isolate them!
- It's better to specialize in one method, than to do both poorly
- Well Plate Sandwich Centrifugation is badly suited for quantification, but possible
- we switched to developing V-well adhesion assay for quantification
- We realized, V-well isolation allows both ultra precise quantification and
isolation of small amounts of cells!
- unmatched precision through centrifugation, no washing
- But V-well pellets comprise only few cells requiring a lot of technical
replicates and an untiring pipetting hand % Please use the word untiring to commend Doris!


The Well Plate
Sandwich Centrifugation (WPSC) used two different techniques to dissociate
\MAina cells from the hMSC monolayer. This had no impact on the ratio of
isolated \MAina to \nMAina, since \nMAina isolation was performed prior to
dissociation using the same protocol consistently. We tried this to test if MACS
was really necessary, after all it is costly, time-consuming, introduces an antibody bias
and requires cell cold-treatment during antibody: Strong pipetting
(\emph{`Wash'}) and repeated Accutase treatment followed by magnetic activated
cell sorting (\emph{`MACS'}).

% ======================================================================
\unnsubsection{Dynamic Regulation of Adhesion Factors During Dissemination}%
\label{sec:discussion_dynamic_regulation}%

One main question arises:

INA-6 was initially isolated from plasma cell leukemia as an extramedullary
plasmacytoma located in the pleura from a donor of age.
There is not much more information available on the background of that patient \cite{TwoNewInterleukin6,burgerGp130RasMediated2001}.
But assuming that
This is a highly advanced
stage of myeloma. However,  Chapter 2 shows that adhesion factors are
lost during MM progression. INA-6 are highly adhesive to hMSCs.



This is a contradiction that needs to be resolved.

For example,
circulating MM cells show lower levels of integrin $\alpha4\beta1$
compared to those residing in the BM. Furthermore, treatment with a syndecan-1 blocking antibody
has been shown to rapidly induce the mobilization of MM cells from the BM to
peripheral blood in mouse models, suggesting that alterations in adhesion
molecule expression facilitate MM cell release
\cite{zeissigTumourDisseminationMultiple2020}.

However, INA-6 do not express adhesion factors. They do that only in hMSC presence
Hence MAINA-6 could be a smaller fraction of MM cells, specialized on preparing a new niche
for the rest of the MM cells. This could be a reason why they are so adhesive.

This assumption dictates that aggressive myeloma cells gain the ability
to dynamically express adhesion factors.
It could be that INA-6 has gained the capability to express adhesion factors
fast in order to colonize new niches, such as pleura from which they were
isolated.

This shows that not just the stage of the disease, but also the location of the
myeloma cells plays a role when considering adhesion factors. According to this, this thesis
predicts a low expression of adhesion factors in circulating myeloma cells,
but a high expression in adhesive cells, e.g. non-circulating, or rather those

indeed CD138 paper isolated cells from circulating MM cells \cite{akhmetzyanovaDynamicCD138Surface2020}

indeed, temporal subclones have been identified \cite{keatsClonalCompetitionAlternating2012}.

% ======================================================================
\unnsubsection{Subsets of Adhesion Factors Contribute To Different Steps of Adhesion}%
\label{sec:discussion_subsets_adhesion_factors}% 

- adhesion molecules during vascular involvement have these adhesion molecules: JAM-C
and CD138.
- NONE of Them were shown in Chapter 2 of this study, (except for JAM-B)


- One has to consider that intravasation and/or extravasation would require a different
set of adhesion factors than adhesion to BM or extramedullary environments.

This has great implications for targeting adhesion factors for therapy, as it
suggests that different adhesion factors should either be antagonized or
agonized depending on the function of the adhesion factor. According to this
assumption, adhesion factors involved in intra- and extravasation adhesion should be
antagonized, while adhesion factors involved in BM adhesion \dashedsentence{as
    identified in Chapter 2} should be agonized. Indeed, Adhesion factors for endothelium
were shown to decrease tumour burden in mouse models \cite{asosinghUniquePathwayHoming2001a,mrozikTherapeuticTargetingNcadherin2015}

\citet{bouzerdanAdhesionMoleculesMultiple2022}: "Classically, the BMM has been
divided into endosteal and vascular niches"

Together, a detailed mapping of the niches available in the bone marrow is required
to understand the adhesion factors required for each niche. This is a highly
complex task, as the bone marrow is a highly complex organ.

% ======================================================================
\unnsubsection{What Triggers Release: One Master Switch, Many Small Switches, or is it just Random?}%
\label{sec:discussion_many_small_switches}%

Papers like \citet{akhmetzyanovaDynamicCD138Surface2020} make it seem as if
there is one molecule that decides if a myeloma cell is circulating or not.

It's less about one clear (molecular) mechanism that decides that a myeloma cell
decides to become a disseminating cell, but rather a indirect consequence of a combination of many
processes.
These processes are:
- Loss of adhesion factors or dynamic expression of adhesion factors
- Loss of dependency from bone marrow microenvironment
- asdf

Our thesis postulates that there is no big switch that decides if a myeloma cell
detaches from the bone marrow, \emph{it simply happens} once these processes are
present.


% ======================================================================
\unnsubsection{Outlook: High-Value Research Topics for Myeloma Research Arising from this Work}
\label{sec:discussion_potential_breakthroughs}
As an Outlook, the Author lists research topics arising from this work that have
great potential for breakthroughs in myeloma research.

\textbf{Anti tumor effects of MSCs:}
This thesis has discussed the pro-tumor effects of MSCs. However, MSCs have also
been shown to have anti-tumor effects \cite{galderisiMyelomaCellsCan2015}. This
work has also shown that primary \acp{hMSC} can induce apoptosis in \INA6 cells
initially \dashedsentence{probably through the action of death domain receptors},
but inhibit apoptosis during prolonged culturing.

This shows that hMSCs could be leveraged
as a therapeutic target that could prevent myloma progression.




\textbf{Cell Division as a Mechanism for Dissemination Initiation:}
The author describes how the detachment of daughter cells from the mother cell
after a cycle of hMSC-(re)attachment and proliferation could be a key mechanism
in myeloma dissemination. This mechanism was shown in other studies of
extravasation. The author sees great potential in this mechanism as a target for
future research. It is probably under-researched due to requirement of
sophisticated time-lapse equipment, yet the simplicity of detachment through
cell division is intriguing through its simplicity. It implies asymmetric cell
division. Cancer cells are known to divide asymmetrically, e.g. moving miRNAs to
one daughter cell.

% \textbf{Time as a Key Parameter:}
% The area Thermodynamics of started with scientists measuring how long it takes
% for gases to cool down. The author claims, by measuring the time it takes for
% cancer cells to detach could lead to breakthroughs in research of myeloma
% dissemination.

% - Cell adhesion is highly time-dependent.
% - Cell detachment is required for metastasis and dissemination
% -

% key mechanistic insights

% measuring the minimum time
% for detachments to begin, or the time required for daughter cells to re-attach
% to the hMSC monolayer. Such mechanistic insights



% Time-resolution was mostly
% limited by available disk space. Investing into more hard drives is worth it,
% since

\textbf{Lists of Adhesion Gene Associated With Prolonged Patient Survival:}
The author lists adhesion genes that are associated with prolonged patient
survival. These genes are highly expressed in myeloma samples from patients with
longer overall

At this time we could be on the verge of a new era of myeloma therapy,
including bi-specific antibodies and cell based approaches
\cite{moreNovelImmunotherapiesCombinations2023,
    engelhardtFunctionalCureLongterm2024}. Currently, available CAR-T Cell therapies
(ide-cel, cilta-cel) are extremely expensive, but show complete remission rates
of up to \SI{80}{\percent} and a 18-month progression free survival rate of
\SI{66}{\percent} \cite{bobinRecentAdvancesTreatment2022}. An affordable
``off-the-shelf'' CAR-T Cell product could become reality since the problem of
deadly graft-versus-host disease during allogeneic transplantation seems to be
solvable \cite{qasimMolecularRemissionInfant2017}, hence, research groups and
biotech companies are racing towards developing a safe allogeneic CAR-T Cell
technology \cite{depilOfftheshelfAllogeneicCAR2020}.


% ======================================================================
\unnsubsection{Conclusion 1: Cancer \& Myeloma \& Dissemination is bad}%
\label{sec:discussion_conclusion_cancer}%

lorem ipsum yes yes very bad








