% ======================================================================
% == Appendix for Chapter 1: Figures
% ======================================================================




% ## Example of including Figures
% > \includeimage[scale][left bottom right top]{
% >     path/to/picture.pdf/.png/.jpg
% > }{caption}{label}


% ## Fig. S1
% \includeimage[0.99][{\fmleft} 1.8in {\fmright} {\fmtop}]{
\includeimage[1][{\fmleft} 1.8in {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/figures8_supplemental1.pdf
}\figcaption[fig:S1]{ %
    Principle and quantification of the V-well adhesion assay of fluorescently
    labeled myeloma cells adapted by
    \citet{weetallHomogeneousFluorometricAssay2001}. \tile{A} Sample: Subsequent
    rounds of centrifugation and removal of cell pellet yielded the size of
    adhesive subpopulations. Fluorescently stained INA-6 cells were added to an
    hMSC monolayer. Non-adherent INA-6 cells (V2) were pelleted in the well-tip.
    Pellets were quantified by fluorescence brightness and isolated by
    pipetting. Immobile INA-6 cells (V3) were manually detached by forceful
    pipetting. Reference: Omitting adhesive hMSC-layer yielded~\SI{100}{\percent}
    non-adherent cells (V1) after the first centrifugation step; Background:
    hMSC monolayer was used as background signal. \tile{B} Calculation of the
    population size relative to total cells starting with pellet intensity. The
    shown example is the pellet gained by centrifuging mobile subpopulation (V2)
    after \SI{1}{\hour} of co-culture. (see \autoref{fig:3} for context): Intensity values
    from pellet images were summarized. After subtracting the unlabeled hMSC
    signal and normalization by a full-size pellet (reference), the resulting
    values represented the fraction of the adhesive subpopulation. \tile{C} One
    of three biological replicates summarized in \autoref{fig:3}. Line range
    shows the standard deviation of four technical replicates. Non.Adh. Rem.:
    Fluorescence signal after removal of~V2. \tile{D} Example images of myeloma
    cell lines (INA-6, U266, MM.1S) pelleted in the tip of V-wells. The leftmost
    image shows the recorded area in a complete V-well. Scale bar = \SI{200}{\um}.
    \tile{E} Results from (D) comparing adhesion strength of three myeloma cell
    lines to hMSC. Error bars represent technical deviation. MM: Multiple
    Myeloma. \selfplagiarismone}
\newpage


% ## Fig. S2
\includeimage[1][{\fmleft} {\fmbottom} {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/figures8_supplemental2.pdf
}\figcaption[fig:S2]{ %
    Validation of image cytometric analysis of cell
    cycle in four INA-6 cultures. \tile{A} Left: Example image cytometric scan: INA-6
    cells were stained with Hoechst33342 and scanned by automated fluorescence
    microscopy. Right: The image was segmented using a convolutional neural
    network (ZEISS ZEN intellesis) trained to discern healthy nuclei (green)
    from fragmented ones (magenta). Doublets are excluded by setting an area-
    and roundness threshold. Scale bar: \SI{20}{\um}. \tile{B} Two example images from the
    training set. \tile{C} Quality of image cytometric data was ensured by plotting
    the distribution of nuclei brightnesses vs. the distribution of both
    nuclei-roundnesses and nuclei-areas. Nuclei with double fluorescence
    intensity have the same roundness while their area increases, as expected
    from a cell in G2 phase. \tile{D} The same samples from (C) were also measured
    with flow cytometry. Representative example of gating strategy: Left: Dead
    cells were excluded by setting a minimum threshold for side-scattering
    (SSC-A). Right: Doublets were excluded by setting a maximum threshold for
    forward scatter area (FSC-A) (sample ``5'' represents culture ``4'' in this
    figure). \tile{E} Cell cycle profiles of four independent INA-6 cultures were
    measured by both image cytometry (top) and flow cytometry (bottom). For both
    methods, frequencies of G0/G1, S, and G2M were summed up by setting
    fluorescence intensity thresholds. \tile{F} Image cytometry yields the same
    frequencies for G0/G1, S, and G2M when compared to flow cytometry. RM-ANOVA
    showed that the method has no significant effect on the frequencies of cell
    cycle populations $[F(1,3)=1.421, p\text{-unc}=.32]$. \tile{G} Results from (F) in tabular
    form. On average, frequencies for G0/G1, S, and G2M measured by Image
    cytometry differ by 0.95 percent points compared to flow cytometry
    measurement. Cult.: Culture; C.: Image cytometry; Abs.: Absolute cell count;
    Rel.: Relative cell count; Diff.: Difference between relative cell counts
    determined by flow cytometry and image cytometry. \selfplagiarismone}
\newpage

% ## Fig. S3
\includeimage[1][{.97\fmleft} {\fmbottom} {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/figures8_supplemental3.pdf
}\figcaption[fig:S3]{ % 
    Cell cycle analysis of INA-6 pellets gained from V-Well Adhesion assay
    (\autoref{fig:3}). \tile{A} Cell cycle profiles of MSC-adhering subpopulations.
    INA-6 cells were synchronized by double thymidine block followed by
    nocodazole. Cell cycle was released directly before addition to hMSCs.
    Histograms were normalized and summed up across all biological replicates
    ($n = 4$). Technical replicates (3) were pooled prior to cell cycle profiling.
    CoCult. = Co-culture duration. Fraction = Adhesion subpopulations. \tile{B}
    Similar figure to Fig. 3C displaying ratio of INA-6 populations (G2/M to
    G0/G1). \tile{Statistics:} Paired t-test (B). \selfplagiarismone}%

\newpage
% \noindent
\vspace{-1cm}
%
% ## Fig. S4
\includeimage[1.04][{.95\fmleft} {\fmbottom} {\fmright} {\fmtop}]{%
    APPENDIX_CHAPTER1/figures8_supplemental4.pdf%
}
\figcaption[fig:S4]{%
    Representative (one of the four independent sample sets as seen in
    \apdxref{apdx:supplemental}{fig:S3}) curve fitting analysis of cell cycle profiles
    generated by Image Cytometry. t8, t9, t10, and t24 refer to 1, 2, 3, and 24
    hours after the addition of INA-6 cells to hMSCs. \selfplagiarismone}
\newpage

% ## Fig. S5
\includeimage[1][{\fmleft} 4.3in {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/figures8_supplemental5.pdf
}\figcaption[fig:S5]{%
    Correlation of RNAseq with qPCR Left: Validation of RNAseq results
    (\autoref{fig:4}) with qPCR showing the $\log_{2}(\text{foldchange
            expression})$ of 18 genes. For qPCR, Datapoints each represent one
    biological replicate ($n = 10$), which is the mean of technical replicates
    ($n = 3$). Bar height represents mean of biological replicates, error bars
    show standard deviation of biological replicates. Right: Correlation between
    qPCR and RNAseq in terms of $\log_{2}(\text{mean foldchange expression per
            gene})$. Each dot represents one gene shown in the barplot to the left.
    Genes measured with qPCR that showed no differential expression in RNAseq
    were set to have a $-\log_{2}(FC)=0$. Shaded area shows the confidence
    interval of linear regression. Correlation coefficient was calculated using
    Spearman's rank. $N = 18$ genes. $FC$: Fold change expression.
    \selfplagiarismone%
}
\newpage


% ## Fig. S6
\includeimage[1][{\fmleft} 2.7in {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/figures8_supplemental6.pdf
}\vspace{-.5cm}
\figcaption[fig:S6]{ %
    Functional enrichment analysis by Metascape using genes that are
    differentially expressed between MSC-interacting subpopulations. Top:
    Upregulated genes. Bottom: Downregulated genes. \selfplagiarismone%
}
\newpage



% ## Fig. S7
\newcommand{\figscale}{0.9} % > Scale of the figure
\newcommand{\figfmbottom}{3in}
% \addpdf*[.70]{bla}{APPENDIX_CHAPTER1/crc-23-0411-s15.pdf}
\includeimage[{\figscale}][{\fmleft} {\figfmbottom} {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/crc-23-0411-s15_1.pdf
}\figcaption[fig:S7]{ %
    Expression levels of adhesion genes that are downregulated and associated
    with survival (p < 0.01). Bone Marrow Plasma Cell (BMPC), Monoclonal
    Gammopathy of Undetermined Significance (MGUS), Smoldering Multiple Myeloma
    (sMM), Multiple Myeloma (MM), Multiple Myeloma Relapse (MMR).
    \selfplagiarismone%
}

% > Fig. S7 (continued)
\includeimagecontinued[{\figscale}][{\fmleft} {\figfmbottom} {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/crc-23-0411-s15_2.pdf
}
\includeimagecontinued[{\figscale}][{\fmleft} {\figfmbottom} {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/crc-23-0411-s15_3.pdf
}
\includeimagecontinued[{\figscale}][{\fmleft} {\figfmbottom} {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/crc-23-0411-s15_4.pdf
}
\includeimagecontinued[{\figscale}][{\fmleft} {\figfmbottom} {\fmright} {\fmtop}]{
    APPENDIX_CHAPTER1/crc-23-0411-s15_5.pdf
}



% ## Fig. S8
% \addpdf*[.70]{bla}{APPENDIX_CHAPTER1/crc-23-0411-s16.pdf}
\includeimage[0.8][{\fmleft} {.5\fmbottom} {\fmright} {.5\fmtop}]{
    APPENDIX_CHAPTER1/crc-23-0411-s16.pdf
}\figcaption[fig:S8]{%
    Expression levels of adhesion genes that are not downregulated and
    associated with survival (p < 0.01). Bone Marrow Plasma Cell (BMPC),
    Monoclonal Gammopathy of Undetermined Significance (MGUS), Smoldering
    Multiple Myeloma (sMM), Multiple Myeloma (MM), Multiple Myeloma Relapse
    (MMR). \selfplagiarismone%
}
% \newpage




%%%%%%% Test References
% V-Well is described in \apdxref{apdx:supplemental}{fig:S1}

% blaaaa \apdxref{apdx:supplemental}{fig:S7}

% blaaaa \apdxref{apdx:supplemental}{fig:S8}

% V-Well is described in \apdxref{apdx:supplemental}{fig:S1}, \ref{fig:S2}

% Image cytometry is described in \autoref{subapdx:sup_figtabs} \autoref{fig:S2}

% Full analysis is shown in \apdxref{subapdx:example_analysis}

